

<!DOCTYPE html>
<html class="writer-html5" lang="en" >
<head>
  <meta charset="utf-8">
  
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  <title>&lt;no title&gt; &mdash; Sigma2/Metacenter documentation  documentation</title>
  

  
  <link rel="stylesheet" href="../../_static/css/theme.css" type="text/css" />
  <link rel="stylesheet" href="../../_static/pygments.css" type="text/css" />

  
  
  
  

  
  <!--[if lt IE 9]>
    <script src="../../_static/js/html5shiv.min.js"></script>
  <![endif]-->
  
    
      <script type="text/javascript" id="documentation_options" data-url_root="../../" src="../../_static/documentation_options.js"></script>
        <script src="../../_static/jquery.js"></script>
        <script src="../../_static/underscore.js"></script>
        <script src="../../_static/doctools.js"></script>
        <script src="../../_static/language_data.js"></script>
        <script async="async" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/latest.js?config=TeX-AMS-MML_HTMLorMML"></script>
        <script type="text/x-mathjax-config">MathJax.Hub.Config({"tex2jax": {"inlineMath": [["\\(", "\\)"]], "displayMath": [["\\[", "\\]"]], "processRefs": false, "processEnvironments": false}})</script>
    
    <script type="text/javascript" src="../../_static/js/theme.js"></script>

    
    <link rel="index" title="Index" href="../../genindex.html" />
    <link rel="search" title="Search" href="../../search.html" /> 
</head>

<body class="wy-body-for-nav">

   
  <div class="wy-grid-for-nav">
    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >
          

          
            <a href="../../index.html" class="icon icon-home" alt="Documentation Home"> Sigma2/Metacenter documentation
          

          
          </a>

          
            
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="../../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        
        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
              
            
            
              <p class="caption"><span class="caption-text">News</span></p>
<ul>
<li class="toctree-l1"><a class="reference external" href="https://opslog.sigma2.no">Latest changes and events</a></li>
<li class="toctree-l1"><a class="reference external" href="https://www.sigma2.no/hardware-status">Hardware live status</a></li>
</ul>
<p class="caption"><span class="caption-text">Getting help</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../../getting_help/support_line.html">Support line</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../getting_help/how_to_write_good_support_requests.html">Writing good support requests</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../getting_help/qa-sessions.html">Open Question &amp; Answer Sessions for All Users</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../getting_help/lost_forgotten_password.html">Lost or expiring password</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../getting_help/project_leader_support.html">Project leader support</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../getting_help/advanced_user_support.html">Advanced User Support (AUS)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../getting_help/publish_research_data.html">Publish Research Data</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../getting_help/course_resources.html">CRaaS - Course Resources as a Service</a></li>
</ul>
<p class="caption"><span class="caption-text">Getting started</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../../frontpage.html">The Norwegian Academic HPC Services</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../getting_started/applying_account.html">How do I get an HPC account?</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../getting_started/applying_resources.html">Applying for computing and storage</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../getting_started/training.html">Training material</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../getting_started/getting_started.html">Getting Started</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../getting_started/create_ssh_keys.html">SSH</a></li>
</ul>
<p class="caption"><span class="caption-text">HPC Machines</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../../hpc_machines/hardware_overview.html">Overview over our machines</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../hpc_machines/betzy.html">Betzy</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../hpc_machines/fram.html">Fram</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../hpc_machines/saga.html">Saga</a></li>
<li class="toctree-l1"><a class="reference external" href="https://hpc-uit.readthedocs.io">Stallo</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../hpc_machines/migration2metacenter.html">Migration to a Metacenter HPC machine</a></li>
</ul>
<p class="caption"><span class="caption-text">Software</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../modulescheme.html">Software Module Scheme</a></li>
<li class="toctree-l1"><a class="reference internal" href="../installed_software.html">Installed Software</a></li>
<li class="toctree-l1"><a class="reference internal" href="../containers.html">Running Singularity and Docker containers</a></li>
<li class="toctree-l1"><a class="reference internal" href="../appguides.html">Application guides</a></li>
<li class="toctree-l1"><a class="reference internal" href="../userinstallsw.html">How can I, as a user, install software for myself or my project with EasyBuild?</a></li>
<li class="toctree-l1"><a class="reference internal" href="../userinstallsw.html#how-can-i-as-user-install-python-packages">How can I as user install Python packages?</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses.html">Licenses and access policies</a></li>
</ul>
<p class="caption"><span class="caption-text">Jobs</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../../jobs/submitting.html">Submitting jobs</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../jobs/choosing_job_types.html">Job Types</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../jobs/job_scripts.html">Job Scripts</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../jobs/running-scientific-software.html">Running scientific software</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../jobs/monitoring.html">Monitoring jobs</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../jobs/choosing_memory_settings.html">How to choose the right amount of memory</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../jobs/common_job_failures.html">Common job failures</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../jobs/performance.html">How to check the performance and scaling using Arm Performance Reports</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../jobs/interactive_jobs.html">Interactive jobs</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../jobs/projects_accounting.html">Projects and accounting</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../jobs/network-access.html">No network access on compute nodes</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../jobs/dos_and_donts.html">Dos and Don’ts</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../jobs/guides.html">Guides</a></li>
</ul>
<p class="caption"><span class="caption-text">Files and Storage</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../../files_storage/nird.html">NIRD - National Infrastructure for Research Data</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../files_storage/clusters.html">Storage areas on HPC clusters</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../files_storage/backup.html">Backup on Betzy, Fram, Saga, and NIRD</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../files_storage/sharing_files.html">Data handling and storage policy</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../files_storage/file_transfer.html">File transfer</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../files_storage/performance.html">Storage performance tuning</a></li>
</ul>
<p class="caption"><span class="caption-text">Code Development</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../../code_development/building.html">Building scientific software</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../code_development/compilers.html">Compilers</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../code_development/performance.html">Performance Analysis and Tuning</a></li>
</ul>

            
          
        </div>
        
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" aria-label="top navigation">
        
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="../../index.html">Sigma2/Metacenter documentation</a>
        
      </nav>


      <div class="wy-nav-content">
        
        <div class="rst-content">
        
          















<div role="navigation" aria-label="breadcrumbs navigation">

  <ul class="wy-breadcrumbs">
    
      <li><a href="../../index.html" class="icon icon-home"></a> &raquo;</li>
        
      <li>&lt;no title&gt;</li>
    
    
      <li class="wy-breadcrumbs-aside">
        
            
            
              <a href="https://github.com/UNINETTSigma2/documentation/blob/master/software/installed_software/saga_modules.md" class="fa fa-github"> Edit on GitHub</a>
            
          
        
      </li>
    
  </ul>

  
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  <p>###ABySS
<strong>Description:</strong> Assembly By Short Sequences
<strong>Homepage:</strong> https://www.bcgsc.ca/platform/bioinfo/software/abyss</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ABySS/2.0.2-gompi-2019a</p></li>
</ul>
</details>
</details>
<p>###AdapterRemoval
<strong>Description:</strong> AdapterRemoval searches for and removes remnant adapter sequences
from High-Throughput Sequencing (HTS) data and (optionally) trims low quality bases
from the 3’ end of reads following adapter removal.
<strong>Homepage:</strong> https://github.com/%/(github_account)s/adapterremoval</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>AdapterRemoval/2.3.1-GCC-8.2.0-2.31.1</p></li>
<li><p>AdapterRemoval/2.3.1-foss-2018b</p></li>
</ul>
</details>
</details>
<p>###ADF
<strong>Description:</strong> ADF is an accurate, parallelized, powerful computational chemistry program to understand and
predict chemical structure and reactivity with density functional theory (DFT).
<strong>Homepage:</strong> http://www.scm.com/ADF/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ADF/2019.103+StaticMKL</p></li>
</ul>
</details>
</details>
<p>###AdmixTools
<strong>Description:</strong> ADMIXTOOLS is a software package that supports formal tests of
whether admixture occurred, and makes it possible to infer admixture proportions and dates
<strong>Homepage:</strong> https://github.com/DReichLab/AdmixTools</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>AdmixTools/5.1-GCC-7.3.0-2.30</p></li>
</ul>
</details>
</details>
<p>###ADMIXTURE
<strong>Description:</strong> ADMIXTURE is a software tool for maximum likelihood estimation of individual ancestries from
multilocus SNP genotype datasets. It uses the same statistical model as STRUCTURE but calculates estimates much
more rapidly using a fast numerical optimization algorithm.
<strong>Homepage:</strong> http://software.genetics.ucla.edu/admixture</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ADMIXTURE/1.3.0</p></li>
</ul>
</details>
</details>
<p>###Amber
<strong>Description:</strong> Amber (originally Assisted Model Building with Energy
Refinement) is software for performing molecular dynamics and structure
prediction.
<strong>Homepage:</strong> http://ambermd.org/amber.html</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Amber/18-foss-2018b-AmberTools-18-patchlevel-10-8</p></li>
<li><p>Amber/18-fosscuda-2018b-AmberTools-18-patchlevel-10-8</p></li>
</ul>
</details>
</details>
<p>###Anaconda2
<strong>Description:</strong> Built to complement the rich, open source Python community,
the Anaconda platform provides an enterprise-ready data analytics platform
that empowers companies to adopt a modern open data science analytics architecture.</p>
<p><strong>Homepage:</strong> https://www.continuum.io/anaconda-overview</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Anaconda2/2019.03</p></li>
</ul>
</details>
</details>
<p>###Anaconda3
<strong>Description:</strong> Built to complement the rich, open source Python community,
the Anaconda platform provides an enterprise-ready data analytics platform
that empowers companies to adopt a modern open data science analytics architecture.</p>
<p><strong>Homepage:</strong> https://www.continuum.io/anaconda-overview</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Anaconda3/2019.03</p></li>
</ul>
</details>
</details>
<p>###Ancestry_HMM
<strong>Description:</strong> A hidden Markov model approach for simultaneously estimating local ancestry.
<strong>Homepage:</strong> https://github.com/russcd/Ancestry_HMM</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Ancestry_HMM/200105-foss-2019a</p></li>
</ul>
</details>
</details>
<p>###angsd
<strong>Description:</strong> Program for analysing NGS data.
<strong>Homepage:</strong> http://www.popgen.dk/angsd</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>angsd/0.925-foss-2018b</p></li>
<li><p>angsd/0.928-GCC-7.3.0-2.30</p></li>
<li><p>angsd/0.931-GCC-8.2.0-2.31.1</p></li>
</ul>
</details>
</details>
<p>###angsd-wrapper
<strong>Description:</strong> ANGSD-wrapper is a utility developed to aid in the analysis of next generation sequencing data
<strong>Homepage:</strong> https://github.com/ANGSD-wrapper/angsd-wrapper</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>angsd-wrapper/190321-GCC-7.3.0-2.30</p></li>
</ul>
</details>
</details>
<p>###ant
<strong>Description:</strong> Apache Ant is a Java library and command-line tool whose mission is to drive processes described in
build files as targets and extension points dependent upon each other. The main known usage of Ant is the build of
Java applications.
<strong>Homepage:</strong> http://ant.apache.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ant/1.10.5-Java-1.8</p></li>
</ul>
</details>
</details>
<p>###ANTLR
<strong>Description:</strong> ANTLR, ANother Tool for Language Recognition, (formerly PCCTS)
is a language tool that provides a framework for constructing recognizers,
compilers, and translators from grammatical descriptions containing
Java, C#, C++, or Python actions.
<strong>Homepage:</strong> http://www.antlr2.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ANTLR/2.7.7-GCCcore-7.3.0</p></li>
</ul>
</details>
</details>
<p>###APR
<strong>Description:</strong> Apache Portable Runtime (APR) libraries.
<strong>Homepage:</strong> http://apr.apache.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>APR/1.6.3-GCCcore-7.3.0</p></li>
<li><p>APR/1.7.0-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###APR-util
<strong>Description:</strong> Apache Portable Runtime (APR) util libraries.
<strong>Homepage:</strong> http://apr.apache.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>APR-util/1.6.1-GCCcore-7.3.0</p></li>
<li><p>APR-util/1.6.1-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###ARKS
<strong>Description:</strong> Scaffolding genome sequence assemblies using 10X Genomics GemCode/Chromium data
<strong>Homepage:</strong> https://github.com/bcgsc/arcs</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ARKS/1.0.4-gompi-2019a</p></li>
</ul>
</details>
</details>
<p>###Arm-PerfReports
<strong>Description:</strong> Single page HTML profiler for parallel programs.
<strong>Homepage:</strong> https://developer.arm.com/products/software-development-tools/hpc</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Arm-PerfReports/19.1</p></li>
</ul>
</details>
</details>
<p>###Armadillo
<strong>Description:</strong> Armadillo is an open-source C++ linear algebra library (matrix maths) aiming towards
a good balance between speed and ease of use. Integer, floating point and complex numbers are supported,
as well as a subset of trigonometric and statistics functions.
<strong>Homepage:</strong> https://arma.sourceforge.net/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Armadillo/9.700.2-foss-2019a</p></li>
</ul>
</details>
</details>
<p>###arpack-ng
<strong>Description:</strong> ARPACK is a collection of Fortran77 subroutines designed to solve large scale eigenvalue problems.
<strong>Homepage:</strong> https://github.com/opencollab/arpack-ng</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>arpack-ng/3.7.0-foss-2019a</p></li>
</ul>
</details>
</details>
<p>###ASE
<strong>Description:</strong> ASE is a python package providing an open source Atomic Simulation Environment
in the Python scripting language.
<strong>Homepage:</strong> https://wiki.fysik.dtu.dk/ase</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ASE/3.17.0-foss-2018b-Python-3.6.6</p></li>
<li><p>ASE/3.17.0-intel-2018b-Python-3.6.6</p></li>
<li><p>ASE/3.18.0-foss-2019a-Python-3.7.2</p></li>
<li><p>ASE/3.18.0-intel-2019a-Python-3.7.2</p></li>
</ul>
</details>
</details>
<p>###AUGUSTUS
<strong>Description:</strong> AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences
<strong>Homepage:</strong> http://bioinf.uni-greifswald.de/augustus/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>AUGUSTUS/3.3.2-intel-2018b-Python-2.7.15</p></li>
</ul>
</details>
</details>
<p>###Autoconf
<strong>Description:</strong> Autoconf is an extensible package of M4 macros that produce shell scripts
to automatically configure software source code packages. These scripts can
adapt the packages to many kinds of UNIX-like systems without manual user
intervention. Autoconf creates a configuration script for a package from a
template file that lists the operating system features that the package can
use, in the form of M4 macro calls.</p>
<p><strong>Homepage:</strong> http://www.gnu.org/software/autoconf/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Autoconf/2.69-GCCcore-6.4.0</p></li>
<li><p>Autoconf/2.69-GCCcore-7.3.0</p></li>
<li><p>Autoconf/2.69-GCCcore-8.2.0</p></li>
<li><p>Autoconf/2.69-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###Automake
<strong>Description:</strong> Automake: GNU Standards-compliant Makefile generator
<strong>Homepage:</strong> http://www.gnu.org/software/automake/automake.html</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Automake/1.15.1-GCCcore-6.4.0</p></li>
<li><p>Automake/1.15.1-GCCcore-7.3.0</p></li>
<li><p>Automake/1.15.1-GCCcore-8.2.0</p></li>
<li><p>Automake/1.16.1-GCCcore-7.3.0</p></li>
<li><p>Automake/1.16.1-GCCcore-8.2.0</p></li>
<li><p>Automake/1.16.1-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###Autotools
<strong>Description:</strong> This bundle collect the standard GNU build tools: Autoconf, Automake
and libtool</p>
<p><strong>Homepage:</strong> http://autotools.io</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Autotools/20170619-GCCcore-6.4.0</p></li>
<li><p>Autotools/20170619-GCCcore-7.3.0</p></li>
<li><p>Autotools/20170619-GCCcore-8.2.0</p></li>
<li><p>Autotools/20180311-GCCcore-7.3.0</p></li>
<li><p>Autotools/20180311-GCCcore-8.2.0</p></li>
<li><p>Autotools/20180311-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###BAMM
<strong>Description:</strong> BAMM is oriented entirely towards detecting and quantifying heterogeneity in evolutionary rates.
It uses reversible jump Markov chain Monte Carlo to automatically explore a vast universe of candidate models of
lineage diversification and trait evolution.
<strong>Homepage:</strong> http://bamm-project.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>BAMM/2.5.0-foss-2018b</p></li>
</ul>
</details>
</details>
<p>###BamTools
<strong>Description:</strong> BamTools provides both a programmer’s API and an end-user’s toolkit for handling BAM files.
<strong>Homepage:</strong> https://github.com/pezmaster31/bamtools</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>BamTools/2.5.1-GCC-8.2.0-2.31.1</p></li>
<li><p>BamTools/2.5.1-foss-2018b</p></li>
<li><p>BamTools/2.5.1-iccifort-2019.1.144-GCC-8.2.0-2.31.1</p></li>
<li><p>BamTools/2.5.1-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###BayeScan
<strong>Description:</strong> BayeScan aims at identifying candidate loci under natural selection from genetic data,
using differences in allele frequencies between populations.
<strong>Homepage:</strong> http://cmpg.unibe.ch/software/BayeScan/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>BayeScan/2.1-foss-2018b</p></li>
<li><p>BayeScan/2.1-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###BayeScEnv
<strong>Description:</strong> BayeScEnv is a Fst-based, genome-scan method that uses environmental variables to detect
local adaptation.
<strong>Homepage:</strong> https://github.com/devillemereuil/bayescenv</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>BayeScEnv/1.1-foss-2018b</p></li>
</ul>
</details>
</details>
<p>###Bazel
<strong>Description:</strong> Bazel is a build tool that builds code quickly and reliably.
It is used to build the majority of Google’s software.
<strong>Homepage:</strong> https://bazel.io/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Bazel/0.18.0-GCCcore-7.3.0</p></li>
<li><p>Bazel/0.20.0-GCCcore-7.3.0</p></li>
<li><p>Bazel/0.29.1-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###BBMap
<strong>Description:</strong> BBMap short read aligner, and other bioinformatic tools.
<strong>Homepage:</strong> https://sourceforge.net/projects/bbmap/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>BBMap/38.50b-GCC-8.2.0-2.31.1</p></li>
</ul>
</details>
</details>
<p>###BCFtools
<strong>Description:</strong> Samtools is a suite of programs for interacting with high-throughput sequencing data.
BCFtools
<strong>Homepage:</strong> http://www.htslib.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>BCFtools/1.9-foss-2018b</p></li>
<li><p>BCFtools/1.9-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###bcolz
<strong>Description:</strong> bcolz provides columnar, chunked data containers that can be compressed either in-memory and on-disk.
Column storage allows for efficiently querying tables, as well as for cheap column addition and removal.
It is based on NumPy, and uses it as the standard data container to communicate with bcolz objects,
but it also comes with support for import/export facilities to/from HDF5/PyTables tables and pandas dataframes.
<strong>Homepage:</strong> http://bcolz.blosc.org/en/latest/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>bcolz/1.2.1-foss-2019a-Python-3.7.2</p></li>
</ul>
</details>
</details>
<p>###Beagle
<strong>Description:</strong> Beagle is a software package for phasing genotypes and for imputing ungenotyped markers
<strong>Homepage:</strong> http://faculty.washington.edu/browning/beagle/beagle.html</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Beagle/4.1-Java-1.8</p></li>
<li><p>Beagle/5.1-Java-11</p></li>
</ul>
</details>
</details>
<p>###beagle-lib
<strong>Description:</strong> beagle-lib is a high-performance library that can perform the core calculations at the heart of most
Bayesian and Maximum Likelihood phylogenetics packages.
<strong>Homepage:</strong> https://github.com/beagle-dev/beagle-lib</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>beagle-lib/3.0.2-foss-2018b</p></li>
<li><p>beagle-lib/3.1.2-GCC-8.2.0-2.31.1</p></li>
</ul>
</details>
</details>
<p>###Beast
<strong>Description:</strong> BEAST is a cross-platform program for Bayesian MCMC analysis of molecular
sequences. It is entirely orientated towards rooted, time-measured phylogenies inferred using
strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies
but is also a framework for testing evolutionary hypotheses without conditioning on a single
tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted
proportional to its posterior probability.
<strong>Homepage:</strong> http://beast2.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Beast/2.5.2-GCC-8.2.0-2.31.1</p></li>
</ul>
</details>
</details>
<p>###BEDTools
<strong>Description:</strong> The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps
and computing coverage. The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF,
and SAM/BAM.
<strong>Homepage:</strong> https://github.com/arq5x/bedtools2</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>BEDTools/2.27.1-foss-2018b</p></li>
<li><p>BEDTools/2.27.1-intel-2018b</p></li>
<li><p>BEDTools/2.28.0-GCC-8.2.0-2.31.1</p></li>
<li><p>BEDTools/2.28.0-iccifort-2019.1.144-GCC-8.2.0-2.31.1</p></li>
</ul>
</details>
</details>
<p>###BEEF
<strong>Description:</strong> BEEF-vdW functionals and Bayesian error estimates to Fortran- or C-based DFT codes.
<strong>Homepage:</strong> https://confluence.slac.stanford.edu/display/SUNCAT/BEEF+Functional+Software</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>BEEF/0.1.1-intel-2019a</p></li>
</ul>
</details>
</details>
<p>###binutils
<strong>Description:</strong> binutils: GNU binary utilities
<strong>Homepage:</strong> http://directory.fsf.org/project/binutils/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>binutils/2.28</p></li>
<li><p>binutils/2.28-GCCcore-6.4.0</p></li>
<li><p>binutils/2.30</p></li>
<li><p>binutils/2.30-GCCcore-7.3.0</p></li>
<li><p>binutils/2.31.1</p></li>
<li><p>binutils/2.31.1-GCCcore-8.2.0</p></li>
<li><p>binutils/2.32</p></li>
<li><p>binutils/2.32-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###BioPerl
<strong>Description:</strong> Bioperl is the product of a community effort to produce Perl code which is useful in biology.
Examples include Sequence objects, Alignment objects and database searching objects.
<strong>Homepage:</strong> http://www.bioperl.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>BioPerl/1.7.2-GCCcore-8.3.0</p></li>
<li><p>BioPerl/1.7.2-foss-2018b-Perl-5.28.0</p></li>
</ul>
</details>
</details>
<p>###Biopython
<strong>Description:</strong> Biopython is a set of freely available tools for biological
computation written in Python by an international team of developers. It is
a distributed collaborative effort to develop Python libraries and
applications which address the needs of current and future work in
bioinformatics.
<strong>Homepage:</strong> http://www.biopython.org</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Biopython/1.73-foss-2019a</p></li>
<li><p>Biopython/1.73-intel-2019a</p></li>
</ul>
</details>
</details>
<p>###Bison
<strong>Description:</strong> Bison is a general-purpose parser generator that converts an annotated
context-free grammar into a deterministic LR or generalized LR (GLR) parser
employing LALR(1) parser tables.</p>
<p><strong>Homepage:</strong> http://www.gnu.org/software/bison</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Bison/3.0.4</p></li>
<li><p>Bison/3.0.4-GCCcore-6.4.0</p></li>
<li><p>Bison/3.0.4-GCCcore-7.3.0</p></li>
<li><p>Bison/3.0.5</p></li>
<li><p>Bison/3.0.5-GCCcore-6.4.0</p></li>
<li><p>Bison/3.0.5-GCCcore-7.3.0</p></li>
<li><p>Bison/3.0.5-GCCcore-8.2.0</p></li>
<li><p>Bison/3.3.2</p></li>
<li><p>Bison/3.3.2-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###BLAST+
<strong>Description:</strong> Basic Local Alignment Search Tool, or BLAST, is an algorithm
for comparing primary biological sequence information, such as the amino-acid
sequences of different proteins or the nucleotides of DNA sequences.
<strong>Homepage:</strong> http://blast.ncbi.nlm.nih.gov/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>BLAST+/2.8.1-foss-2018b</p></li>
<li><p>BLAST+/2.8.1-intel-2018b</p></li>
<li><p>BLAST+/2.9.0-gompi-2019a</p></li>
<li><p>BLAST+/2.9.0-gompi-2019b</p></li>
<li><p>BLAST+/2.9.0-iimpi-2019a</p></li>
</ul>
</details>
</details>
<p>###BLAT
<strong>Description:</strong> BLAT on DNA is designed to quickly find sequences of 95% and greater similarity of length 25 bases or
more.
<strong>Homepage:</strong> https://genome.ucsc.edu/FAQ/FAQblat.html</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>BLAT/3.5-GCC-8.2.0-2.31.1</p></li>
</ul>
</details>
</details>
<p>###BlobTools
<strong>Description:</strong> A modular command-line solution for visualisation,
quality control and taxonomic partitioning of genome datasets.
<strong>Homepage:</strong> https://blobtools.readme.io/docs</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>BlobTools/20180528-foss-2018b-Python-2.7.15</p></li>
</ul>
</details>
</details>
<p>###Blosc
<strong>Description:</strong> Blosc, an extremely fast, multi-threaded, meta-compressor library
<strong>Homepage:</strong> http://www.blosc.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Blosc/1.17.0-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###bokeh
<strong>Description:</strong> Statistical and novel interactive HTML plots for Python
<strong>Homepage:</strong> http://github.com/bokeh/bokeh</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>bokeh/1.3.4-foss-2019a-Python-3.7.2</p></li>
</ul>
</details>
</details>
<p>###Boost
<strong>Description:</strong> Boost provides free peer-reviewed portable C++ source libraries.
<strong>Homepage:</strong> https://www.boost.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Boost/1.65.1-intel-2018b</p></li>
<li><p>Boost/1.67.0-foss-2018b</p></li>
<li><p>Boost/1.67.0-fosscuda-2018b</p></li>
<li><p>Boost/1.67.0-intel-2018b</p></li>
<li><p>Boost/1.68.0-foss-2018b-Python-2.7.15</p></li>
<li><p>Boost/1.68.0-foss-2018b-Python-3.6.6</p></li>
<li><p>Boost/1.68.0-intel-2018b-Python-2.7.15</p></li>
<li><p>Boost/1.68.0-intel-2018b-Python-3.6.6</p></li>
<li><p>Boost/1.70.0-gompi-2019a</p></li>
<li><p>Boost/1.70.0-iimpi-2019a</p></li>
<li><p>Boost/1.71.0-gompi-2019b</p></li>
<li><p>Boost/1.71.0-iimpi-2019b</p></li>
</ul>
</details>
</details>
<p>###Boost.Python
<strong>Description:</strong> Boost.Python is a C++ library which enables seamless interoperability between C++
and the Python programming language.
<strong>Homepage:</strong> http://boostorg.github.io/python</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Boost.Python/1.67.0-foss-2018b-Python-2.7.15</p></li>
<li><p>Boost.Python/1.67.0-foss-2018b-Python-3.6.6</p></li>
<li><p>Boost.Python/1.67.0-fosscuda-2018b-Python-2.7.15</p></li>
<li><p>Boost.Python/1.67.0-intel-2018b-Python-2.7.15</p></li>
<li><p>Boost.Python/1.67.0-intel-2018b-Python-3.6.6</p></li>
<li><p>Boost.Python/1.70.0-gompi-2019a</p></li>
<li><p>Boost.Python/1.70.0-iimpi-2019a</p></li>
</ul>
</details>
</details>
<p>###Bowtie
<strong>Description:</strong> Bowtie is an ultrafast, memory-efficient short read aligner.
It aligns short DNA sequences (reads) to the human genome.
<strong>Homepage:</strong> http://bowtie-bio.sourceforge.net/index.shtml</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Bowtie/1.2.2-foss-2018b</p></li>
</ul>
</details>
</details>
<p>###Bowtie2
<strong>Description:</strong> Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads
to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s
of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes.
Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome,
its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.
<strong>Homepage:</strong> http://bowtie-bio.sourceforge.net/bowtie2/index.shtml</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Bowtie2/2.3.4.2-foss-2018b</p></li>
<li><p>Bowtie2/2.3.4.2-intel-2018b</p></li>
<li><p>Bowtie2/2.3.5.1-GCC-8.2.0-2.31.1</p></li>
<li><p>Bowtie2/2.3.5.1-iccifort-2019.1.144-GCC-8.2.0-2.31.1</p></li>
</ul>
</details>
</details>
<p>###BreakDancer
<strong>Description:</strong> BreakDancer is a Perl/C++ package that provides genome-wide detection of structural variants from
next generation paired-end sequencing reads
<strong>Homepage:</strong> http://gmt.genome.wustl.edu/packages/breakdancer</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>BreakDancer/1.4.5-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###BSMAP
<strong>Description:</strong> BSMAP is a short reads mapping software for bisulfite sequencing reads.
<strong>Homepage:</strong> https://code.google.com/p/bsmap/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>BSMAP/2.90-foss-2018b</p></li>
</ul>
</details>
</details>
<p>###BUSCO
<strong>Description:</strong> BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs
<strong>Homepage:</strong> http://busco.ezlab.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>BUSCO/3.0.2-intel-2018b-Python-2.7.15</p></li>
</ul>
</details>
</details>
<p>###BWA
<strong>Description:</strong> Burrows-Wheeler Aligner (BWA) is an efficient program that aligns
relatively short nucleotide sequences against a long reference sequence such as the human genome.
<strong>Homepage:</strong> http://bio-bwa.sourceforge.net/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>BWA/0.7.17-GCC-8.2.0-2.31.1</p></li>
<li><p>BWA/0.7.17-foss-2018b</p></li>
<li><p>BWA/0.7.17-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###bzip2
<strong>Description:</strong> bzip2 is a freely available, patent free, high-quality data compressor. It
typically compresses files to within 10% to 15% of the best available
techniques (the PPM family of statistical compressors), whilst being around
twice as fast at compression and six times faster at decompression.</p>
<p><strong>Homepage:</strong> https://sourceware.org/bzip2</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>bzip2/1.0.6-GCCcore-7.3.0</p></li>
<li><p>bzip2/1.0.6-GCCcore-8.2.0</p></li>
<li><p>bzip2/1.0.8-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###cairo
<strong>Description:</strong> Cairo is a 2D graphics library with support for multiple output devices.
Currently supported output targets include the X Window System (via both Xlib and XCB), Quartz, Win32, image buffers,
PostScript, PDF, and SVG file output. Experimental backends include OpenGL, BeOS, OS/2, and DirectFB
<strong>Homepage:</strong> https://cairographics.org</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>cairo/1.14.12-GCCcore-7.3.0</p></li>
<li><p>cairo/1.16.0-GCCcore-8.2.0</p></li>
<li><p>cairo/1.16.0-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###canu
<strong>Description:</strong> Canu is a fork of the Celera Assembler designed for high-noise single-molecule sequencing
<strong>Homepage:</strong> http://canu.readthedocs.io</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>canu/1.8-GCCcore-8.2.0-Perl-5.28.1</p></li>
<li><p>canu/1.8-foss-2018b-Perl-5.28.0</p></li>
</ul>
</details>
</details>
<p>###CD-HIT
<strong>Description:</strong> CD-HIT is a very widely used program for clustering and
comparing protein or nucleotide sequences.
<strong>Homepage:</strong> http://weizhong-lab.ucsd.edu/cd-hit/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>CD-HIT/4.8.1-foss-2018b</p></li>
<li><p>CD-HIT/4.8.1-foss-2019a</p></li>
<li><p>CD-HIT/4.8.1-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###CDO
<strong>Description:</strong> CDO is a collection of command line Operators to manipulate and analyse Climate and NWP model Data.
<strong>Homepage:</strong> https://code.zmaw.de/projects/cdo</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>CDO/1.9.5-intel-2018b</p></li>
<li><p>CDO/1.9.8-intel-2019b</p></li>
</ul>
</details>
</details>
<p>###CFITSIO
<strong>Description:</strong> CFITSIO is a library of C and Fortran subroutines for reading and writing data files in
FITS (Flexible Image Transport System) data format.
<strong>Homepage:</strong> https://heasarc.gsfc.nasa.gov/fitsio/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>CFITSIO/3.47-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###cftime
<strong>Description:</strong> Time-handling functionality from netcdf4-python
<strong>Homepage:</strong> https://github.com/Unidata/cftime</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>cftime/1.0.1-foss-2018b-Python-3.6.6</p></li>
<li><p>cftime/1.0.1-intel-2018b-Python-3.6.6</p></li>
</ul>
</details>
</details>
<p>###CGAL
<strong>Description:</strong> The goal of the CGAL Open Source Project is to provide easy access to efficient
and reliable geometric algorithms in the form of a C++ library.
<strong>Homepage:</strong> http://www.cgal.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>CGAL/4.11.1-foss-2018b-Python-2.7.15</p></li>
<li><p>CGAL/4.11.1-foss-2018b-Python-3.6.6</p></li>
</ul>
</details>
</details>
<p>###CheckM
<strong>Description:</strong> CheckM provides a set of tools for assessing the quality of genomes recovered from isolates,
single cells, or metagenomes.
<strong>Homepage:</strong> https://github.com/Ecogenomics/CheckM</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>CheckM/1.0.13-foss-2018b-Python-2.7.15</p></li>
<li><p>CheckM/1.0.18-foss-2019a-Python-2.7.15</p></li>
</ul>
</details>
</details>
<p>###Circlator
<strong>Description:</strong> A tool to circularize genome assemblies.
<strong>Homepage:</strong> https://sanger-pathogens.github.io/circlator</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Circlator/1.5.5-foss-2018b-Python-3.6.6</p></li>
</ul>
</details>
</details>
<p>###Clang
<strong>Description:</strong> C, C++, Objective-C compiler, based on LLVM.  Does not
include C++ standard library – use libstdc++ from GCC.
<strong>Homepage:</strong> https://clang.llvm.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Clang/9.0.1-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###CMake
<strong>Description:</strong> CMake, the cross-platform, open-source build system.  CMake is a family of
tools designed to build, test and package software.</p>
<p><strong>Homepage:</strong> http://www.cmake.org</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>CMake/3.9.1-GCCcore-7.3.0</p></li>
<li><p>CMake/3.9.4-GCCcore-7.3.0</p></li>
<li><p>CMake/3.11.4-GCCcore-7.3.0</p></li>
<li><p>CMake/3.12.1</p></li>
<li><p>CMake/3.12.1-GCCcore-7.3.0</p></li>
<li><p>CMake/3.13.3-GCCcore-8.2.0</p></li>
<li><p>CMake/3.15.3-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###CP2K
<strong>Description:</strong> CP2K is a freely available (GPL) program, written in Fortran 95, to perform atomistic and molecular
simulations of solid state, liquid, molecular and biological systems. It provides a general framework for different
methods such as e.g. density functional theory (DFT) using a mixed Gaussian and plane waves approach (GPW), and
classical pair and many-body potentials.
<strong>Homepage:</strong> http://www.cp2k.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>CP2K/6.1-foss-2019a</p></li>
<li><p>CP2K/6.1-intel-2018a</p></li>
</ul>
</details>
</details>
<p>###CUDA
<strong>Description:</strong> CUDA (formerly Compute Unified Device Architecture) is a parallel
computing platform and programming model created by NVIDIA and implemented by the
graphics processing units (GPUs) that they produce. CUDA gives developers access
to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs.
<strong>Homepage:</strong> https://developer.nvidia.com/cuda-toolkit</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>CUDA/9.2.88-GCC-7.3.0-2.30</p></li>
<li><p>CUDA/10.0.130</p></li>
<li><p>CUDA/10.1.105</p></li>
<li><p>CUDA/10.1.105-GCC-8.2.0-2.31.1</p></li>
<li><p>CUDA/10.1.105-iccifort-2019.1.144-GCC-8.2.0-2.31.1</p></li>
<li><p>CUDA/10.1.243</p></li>
<li><p>CUDA/10.1.243-GCC-8.3.0</p></li>
<li><p>CUDA/10.1.243-iccifort-2019.5.281</p></li>
</ul>
</details>
</details>
<p>###cuDNN
<strong>Description:</strong> The NVIDIA CUDA Deep Neural Network library (cuDNN) is
a GPU-accelerated library of primitives for deep neural networks.
<strong>Homepage:</strong> https://developer.nvidia.com/cudnn</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>cuDNN/7.1.4.18-fosscuda-2018b</p></li>
<li><p>cuDNN/7.4.2.24-CUDA-10.0.130</p></li>
<li><p>cuDNN/7.6.4.38</p></li>
<li><p>cuDNN/7.6.4.38-CUDA-10.1.105</p></li>
<li><p>cuDNN/7.6.4.38-CUDA-10.1.243</p></li>
<li><p>cuDNN/7.6.4.38-gcccuda-2019a</p></li>
<li><p>cuDNN/7.6.4.38-gcccuda-2019b</p></li>
</ul>
</details>
</details>
<p>###Cufflinks
<strong>Description:</strong> Transcript assembly, differential expression, and differential regulation for RNA-Seq
<strong>Homepage:</strong> http://cole-trapnell-lab.github.io/cufflinks/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Cufflinks/20190706-gompi-2019a</p></li>
</ul>
</details>
</details>
<p>###cURL
<strong>Description:</strong> libcurl is a free and easy-to-use client-side URL transfer library,
supporting DICT, FILE, FTP, FTPS, Gopher, HTTP, HTTPS, IMAP, IMAPS, LDAP,
LDAPS, POP3, POP3S, RTMP, RTSP, SCP, SFTP, SMTP, SMTPS, Telnet and TFTP.
libcurl supports SSL certificates, HTTP POST, HTTP PUT, FTP uploading, HTTP
form based upload, proxies, cookies, user+password authentication (Basic,
Digest, NTLM, Negotiate, Kerberos), file transfer resume, http proxy tunneling
and more.</p>
<p><strong>Homepage:</strong> https://curl.haxx.se</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>cURL/7.60.0-GCCcore-7.3.0</p></li>
<li><p>cURL/7.63.0-GCCcore-8.2.0</p></li>
<li><p>cURL/7.66.0-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###cutadapt
<strong>Description:</strong> Cutadapt finds and removes adapter sequences, primers, poly-A tails and
other types of unwanted sequence from your high-throughput sequencing reads.
<strong>Homepage:</strong> http://opensource.scilifelab.se/projects/cutadapt/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>cutadapt/1.18-foss-2018b-Python-3.6.6</p></li>
</ul>
</details>
</details>
<p>###Cython
<strong>Description:</strong> Cython is an optimising static compiler for both the Python programming
language and the extended Cython programming language (based on Pyrex).</p>
<p><strong>Homepage:</strong> https://cython.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Cython/0.27.3-GCCcore-8.2.0-Python-2.7.15</p></li>
</ul>
</details>
</details>
<p>###dadi
<strong>Description:</strong> ∂a∂i implements methods for demographic history and selection inference from genetic data,
based on diffusion approximations to the allele frequency spectrum.
<strong>Homepage:</strong> https://bitbucket.org/gutenkunstlab/dadi</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>dadi/2.0.1-intel-2019a-Python-3.7.2</p></li>
</ul>
</details>
</details>
<p>###dask
<strong>Description:</strong> Dask natively scales Python. Dask provides advanced parallelism for analytics, enabling performance
at scale for the tools you love.
<strong>Homepage:</strong> https://dask.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>dask/2.3.0-foss-2019a-Python-3.7.2</p></li>
</ul>
</details>
</details>
<p>###DBus
<strong>Description:</strong> D-Bus is a message bus system, a simple way for applications to talk
to one another.  In addition to interprocess communication, D-Bus helps
coordinate process lifecycle; it makes it simple and reliable to code
a “single instance” application or daemon, and to launch applications
and daemons on demand when their services are needed.</p>
<p><strong>Homepage:</strong> http://dbus.freedesktop.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>DBus/1.13.6-GCCcore-7.3.0</p></li>
<li><p>DBus/1.13.8-GCCcore-8.2.0</p></li>
<li><p>DBus/1.13.12-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###deepTools
<strong>Description:</strong> deepTools is a suite of python tools particularly developed for the efficient analysis of
high-throughput sequencing data, such as ChIP-seq, RNA-seq or MNase-seq.
<strong>Homepage:</strong> https://deeptools.readthedocs.io/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>deepTools/3.3.0-intel-2019a-Python-3.7.2</p></li>
</ul>
</details>
</details>
<p>###DendroPy
<strong>Description:</strong> A Python library for phylogenetics and phylogenetic computing:
reading, writing, simulation, processing and manipulation of phylogenetic trees
(phylogenies) and characters.
<strong>Homepage:</strong> https://pypi.python.org/pypi/DendroPy/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>DendroPy/4.4.0-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###DFT-D3
<strong>Description:</strong> DFT-D3 implements a dispersion correction for density functionals, Hartree-Fock and semi-empirical
quantum chemical methods.
<strong>Homepage:</strong> http://www.thch.uni-bonn.de/tc/index.php?section=downloads&amp;subsection=DFT-D3&amp;lang=english</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>DFT-D3/3.2.0-intel-2019a</p></li>
</ul>
</details>
</details>
<p>###DIAMOND
<strong>Description:</strong> DIAMOND is a sequence aligner for protein and translated DNA searches, designed for high performance
analysis of big sequence data.
<strong>Homepage:</strong> https://github.com/bbuchfink/diamond</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>DIAMOND/0.9.24-iccifort-2019.1.144-GCC-8.2.0-2.31.1</p></li>
</ul>
</details>
</details>
<p>###double-conversion
<strong>Description:</strong> Efficient binary-decimal and decimal-binary conversion routines for IEEE doubles.
<strong>Homepage:</strong> https://github.com/google/double-conversion</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>double-conversion/3.1.4-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###Doxygen
<strong>Description:</strong> Doxygen is a documentation system for C++, C, Java, Objective-C, Python,
IDL (Corba and Microsoft flavors), Fortran, VHDL, PHP, C#, and to some
extent D.</p>
<p><strong>Homepage:</strong> https://www.doxygen.org</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Doxygen/1.8.13-GCCcore-7.3.0</p></li>
<li><p>Doxygen/1.8.14-GCCcore-7.3.0</p></li>
<li><p>Doxygen/1.8.15-GCCcore-8.2.0</p></li>
<li><p>Doxygen/1.8.16-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###EasyBuild
<strong>Description:</strong> EasyBuild is a software build and installation framework
written in Python that allows you to install software in a structured,
repeatable and robust way.
<strong>Homepage:</strong> https://easybuilders.github.io/easybuild</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>EasyBuild/3.9.2</p></li>
<li><p>EasyBuild/3.9.3</p></li>
<li><p>EasyBuild/3.9.4</p></li>
<li><p>EasyBuild/4.0.0</p></li>
<li><p>EasyBuild/4.0.1</p></li>
<li><p>EasyBuild/4.1.0</p></li>
<li><p>EasyBuild/4.1.1</p></li>
</ul>
</details>
</details>
<p>###ecCodes
<strong>Description:</strong> ecCodes is a package developed by ECMWF which provides an application programming interface and
a set of tools for decoding and encoding messages in the following formats: WMO FM-92 GRIB edition 1 and edition 2,
WMO FM-94 BUFR edition 3 and edition 4, WMO GTS abbreviated header (only decoding).
<strong>Homepage:</strong> https://software.ecmwf.int/wiki/display/ECC/ecCodes+Home</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ecCodes/2.9.2-intel-2018b</p></li>
<li><p>ecCodes/2.15.0-iimpi-2019b</p></li>
</ul>
</details>
</details>
<p>###ecoPCR
<strong>Description:</strong> A tool for estimating DNA Barcode primers quality.
<strong>Homepage:</strong> https://git.metabarcoding.org/obitools/ecopcr/wikis/home</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ecoPCR/1.0.1-foss-2019a-Python-2.7.15</p></li>
</ul>
</details>
</details>
<p>###ecoPrimers
<strong>Description:</strong> ecoPrimers is a software that finds primers from a set of sequences
<strong>Homepage:</strong> https://git.metabarcoding.org/obitools/ecoprimers/wikis/home</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ecoPrimers/0.5-GCC-7.3.0-2.30</p></li>
</ul>
</details>
</details>
<p>###Eigen
<strong>Description:</strong> Eigen is a C++ template library for linear algebra: matrices, vectors, numerical solvers,
and related algorithms.
<strong>Homepage:</strong> http://eigen.tuxfamily.org/index.php?title=Main_Page</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Eigen/3.3.7</p></li>
</ul>
</details>
</details>
<p>###EIGENSOFT
<strong>Description:</strong> The EIGENSOFT package combines functionality from our population genetics methods (Patterson et al.
2006)  and our EIGENSTRAT stratification correction method (Price et al. 2006). The EIGENSTRAT method uses principal
components  analysis to explicitly model ancestry differences between cases and controls along continuous axes of
variation;  the resulting correction is specific to a candidate marker’s variation in frequency across ancestral
populations,  minimizing spurious associations while maximizing power to detect true associations. The EIGENSOFT
package has a built-in plotting script and supports multiple file formats and quantitative phenotypes.
<strong>Homepage:</strong> http://www.hsph.harvard.edu/alkes-price/software/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>EIGENSOFT/7.2.1-foss-2018b</p></li>
<li><p>EIGENSOFT/7.2.1-foss-2019a</p></li>
<li><p>EIGENSOFT/7.2.1-intel-2018b</p></li>
<li><p>EIGENSOFT/7.2.1-intel-2019a</p></li>
</ul>
</details>
</details>
<p>###ELPA
<strong>Description:</strong> Eigenvalue SoLvers for Petaflop-Applications .
<strong>Homepage:</strong> http://elpa.rzg.mpg.de</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ELPA/2016.11.001.pre-intel-2018b</p></li>
<li><p>ELPA/2017.11.001-foss-2019a</p></li>
<li><p>ELPA/2017.11.001-intel-2018a</p></li>
<li><p>ELPA/2018.11.001-intel-2019a</p></li>
</ul>
</details>
</details>
<p>###Emacs
<strong>Description:</strong> GNU Emacs is an extensible, customizable text editor–and more.
At its core is an interpreter for Emacs Lisp, a dialect of the Lisp programming
language with extensions to support text editing.
<strong>Homepage:</strong> http://www.gnu.org/software/emacs/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Emacs/25.3-GCCcore-7.3.0</p></li>
</ul>
</details>
</details>
<p>###EMBOSS
<strong>Description:</strong> EMBOSS is ‘The European Molecular Biology Open Software Suite’.
EMBOSS is a free Open Source software analysis package specially developed for
the needs of the molecular biology (e.g. EMBnet) user community.
<strong>Homepage:</strong> http://emboss.sourceforge.net/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>EMBOSS/6.6.0-foss-2018b</p></li>
<li><p>EMBOSS/6.6.0-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###ExaML
<strong>Description:</strong> Exascale Maximum Likelihood (ExaML) code for phylogenetic inference using MPI.
<strong>Homepage:</strong> https://github.com/amkozlov/ExaML</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ExaML/3.0.21-foss-2019a</p></li>
</ul>
</details>
</details>
<p>###Exonerate
<strong>Description:</strong> Exonerate is a generic tool for pairwise sequence comparison.
It allows you to align sequences using a many alignment models, using either
exhaustive dynamic programming, or a variety of heuristics.
<strong>Homepage:</strong> https://www.ebi.ac.uk/about/vertebrate-genomics/software/exonerate</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Exonerate/2.4.0-GCC-8.2.0-2.31.1</p></li>
</ul>
</details>
</details>
<p>###expat
<strong>Description:</strong> Expat is an XML parser library written in C. It is a stream-oriented parser
in which an application registers handlers for things the parser might find
in the XML document (like start tags)</p>
<p><strong>Homepage:</strong> https://libexpat.github.io</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>expat/2.2.5-GCCcore-7.3.0</p></li>
<li><p>expat/2.2.6-GCCcore-8.2.0</p></li>
<li><p>expat/2.2.7-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###extractPIRs
<strong>Description:</strong> Tool to extract the PIRs from BAM files that relies on the samtools API for efficiency
<strong>Homepage:</strong> https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>extractPIRs/1.r68.x86_64</p></li>
</ul>
</details>
</details>
<p>###FastME
<strong>Description:</strong> FastME: a comprehensive, accurate and fast distance-based phylogeny inference program.
<strong>Homepage:</strong> http://www.atgc-montpellier.fr/fastme/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>FastME/2.1.6.1-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###fastNGSadmix
<strong>Description:</strong> fastNGSadmix is a program for infering admixture proportions and doing PCA with a single NGS sample
<strong>Homepage:</strong> https://github.com/e-jorsboe/fastNGSadmix</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>fastNGSadmix/190518-foss-2019a-R-3.6.0</p></li>
</ul>
</details>
</details>
<p>###FastQC
<strong>Description:</strong> FastQC is a quality control application for high throughput
sequence data. It reads in sequence data in a variety of formats and can either
provide an interactive application to review the results of several different
QC checks, or create an HTML based report which can be integrated into a
pipeline.
<strong>Homepage:</strong> http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>FastQC/0.11.8-Java-1.8</p></li>
</ul>
</details>
</details>
<p>###fastsimcoal2
<strong>Description:</strong> Fast sequential markov coalescent simulation of genomic data under complex evolutionary models
<strong>Homepage:</strong> http://cmpg.unibe.ch/software/fastsimcoal2/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>fastsimcoal2/6.0.3-linux64</p></li>
</ul>
</details>
</details>
<p>###FastTree
<strong>Description:</strong> FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide
or protein sequences. FastTree can handle alignments with up to a million of sequences in a reasonable amount of
time and memory.
<strong>Homepage:</strong> http://www.microbesonline.org/fasttree/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>FastTree/2.1.10-foss-2018b</p></li>
<li><p>FastTree/2.1.10-intel-2018b</p></li>
<li><p>FastTree/2.1.11-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###FASTX-Toolkit
<strong>Description:</strong> The FASTX-Toolkit is a collection of command line tools for
Short-Reads FASTA/FASTQ files preprocessing.
<strong>Homepage:</strong> http://hannonlab.cshl.edu/fastx_toolkit/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>FASTX-Toolkit/0.0.14-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###FFC
<strong>Description:</strong> The FEniCS Form Compiler (FFC) is a compiler for finite element variational forms.
<strong>Homepage:</strong> https://bitbucket.org/fenics-project/ffc</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>FFC/2018.1.0-foss-2018b-Python-3.6.6</p></li>
</ul>
</details>
</details>
<p>###FFTW
<strong>Description:</strong> FFTW is a C subroutine library for computing the discrete Fourier transform (DFT)
in one or more dimensions, of arbitrary input size, and of both real and complex data.
<strong>Homepage:</strong> http://www.fftw.org</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>FFTW/3.3.7-intel-2018a</p></li>
<li><p>FFTW/3.3.8-gompi-2018b</p></li>
<li><p>FFTW/3.3.8-gompi-2019a</p></li>
<li><p>FFTW/3.3.8-gompi-2019b</p></li>
<li><p>FFTW/3.3.8-gompic-2018b</p></li>
<li><p>FFTW/3.3.8-intel-2018b</p></li>
<li><p>FFTW/3.3.8-intel-2019a</p></li>
<li><p>FFTW/3.3.8-intel-2019b</p></li>
</ul>
</details>
</details>
<p>###FIAT
<strong>Description:</strong> The FInite element Automatic Tabulator (FIAT) supports generation of arbitrary order
instances of the Lagrange elements on lines, triangles, and tetrahedra. It is also capable of generating
arbitrary order instances of Jacobi-type quadrature rules on the same element shapes.
<strong>Homepage:</strong> https://bitbucket.org/fenics-project/fiat</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>FIAT/2018.1.0-foss-2018b-Python-3.6.6</p></li>
</ul>
</details>
</details>
<p>###file
<strong>Description:</strong> The file command is ‘a file type guesser’, that is, a command-line tool
that tells you in words what kind of data a file contains.
<strong>Homepage:</strong> http://www.darwinsys.com/file/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>file/5.35-GCCcore-7.3.0</p></li>
</ul>
</details>
</details>
<p>###fineRADstructure
<strong>Description:</strong> A package for population structure inference from RAD-seq data
<strong>Homepage:</strong> http://cichlid.gurdon.cam.ac.uk/fineRADstructure.html</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>fineRADstructure/0.3.2r109-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###fineSTRUCTURE
<strong>Description:</strong> fineSTRUCTURE is a fast and powerful algorithm for identifying population structure using
dense sequencing data.
<strong>Homepage:</strong> https://people.maths.bris.ac.uk/~madjl/finestructure/finestructure_info.html</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>fineSTRUCTURE/2.1.3-GCC-7.3.0-2.30</p></li>
</ul>
</details>
</details>
<p>###flex
<strong>Description:</strong> Flex (Fast Lexical Analyzer) is a tool for generating scanners. A scanner,
sometimes called a tokenizer, is a program which recognizes lexical patterns
in text.</p>
<p><strong>Homepage:</strong> http://flex.sourceforge.net/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>flex/2.6.3</p></li>
<li><p>flex/2.6.4</p></li>
<li><p>flex/2.6.4-GCCcore-6.4.0</p></li>
<li><p>flex/2.6.4-GCCcore-7.3.0</p></li>
<li><p>flex/2.6.4-GCCcore-8.2.0</p></li>
<li><p>flex/2.6.4-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###FLUENT
<strong>Description:</strong> ANSYS FLUENT software contains the broad physical modeling capabilities needed
to model flow, turbulence, heat transfer, and reactions for industrial applications ranging from
air flow over an aircraft wing to combustion in a furnace, from bubble columns to oil platforms,
from blood flow to semiconductor manufacturing, and from clean room design to wastewater treatment plants.
<strong>Homepage:</strong> http://www.ansys.com/Products/Simulation+Technology/Fluid+Dynamics/Fluid+Dynamics+Products/ANSYS+Fluent</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>FLUENT/19.4</p></li>
</ul>
</details>
</details>
<p>###fontconfig
<strong>Description:</strong> Fontconfig is a library designed to provide system-wide font configuration,
customization and application access.</p>
<p><strong>Homepage:</strong> https://www.freedesktop.org/wiki/Software/fontconfig/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>fontconfig/2.13.0-GCCcore-7.3.0</p></li>
<li><p>fontconfig/2.13.1-GCCcore-8.2.0</p></li>
<li><p>fontconfig/2.13.1-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###foss
<strong>Description:</strong> GNU Compiler Collection (GCC) based compiler toolchain, including
OpenMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.
<strong>Homepage:</strong> https://easybuild.readthedocs.io/en/master/Common-toolchains.html#foss-toolchain</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>foss/2018b</p></li>
<li><p>foss/2019a</p></li>
<li><p>foss/2019b</p></li>
</ul>
</details>
</details>
<p>###fosscuda
<strong>Description:</strong> GCC based compiler toolchain <strong>with CUDA support</strong>, and including
OpenMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.
<strong>Homepage:</strong> (none)</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>fosscuda/2018b</p></li>
</ul>
</details>
</details>
<p>###freebayes
<strong>Description:</strong> A haplotype-based variant detector.
<strong>Homepage:</strong> https://github.com/ekg/freebayes</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>freebayes/200109-foss-2019a-Python-3.7.2</p></li>
</ul>
</details>
</details>
<p>###freeglut
<strong>Description:</strong> freeglut is a completely OpenSourced alternative to the OpenGL Utility Toolkit (GLUT) library.
<strong>Homepage:</strong> http://freeglut.sourceforge.net/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>freeglut/3.0.0-GCCcore-8.2.0</p></li>
<li><p>freeglut/3.0.0-foss-2018b</p></li>
</ul>
</details>
</details>
<p>###FreeSurfer
<strong>Description:</strong> FreeSurfer is a set of tools for analysis and visualization of structural and functional brain imaging data.
FreeSurfer contains a fully automatic structural imaging stream for processing cross sectional and longitudinal data.
<strong>Homepage:</strong> https://surfer.nmr.mgh.harvard.edu/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>FreeSurfer/6.0.1-centos6_x86_64</p></li>
</ul>
</details>
</details>
<p>###freetype
<strong>Description:</strong> FreeType 2 is a software font engine that is designed to be small, efficient,
highly customizable, and portable while capable of producing high-quality
output (glyph images). It can be used in graphics libraries, display servers,
font conversion tools, text image generation tools, and many other products
as well.</p>
<p><strong>Homepage:</strong> http://freetype.org</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>freetype/2.9.1-GCCcore-7.3.0</p></li>
<li><p>freetype/2.9.1-GCCcore-8.2.0</p></li>
<li><p>freetype/2.10.1-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###FriBidi
<strong>Description:</strong> The Free Implementation of the Unicode Bidirectional Algorithm.</p>
<p><strong>Homepage:</strong> https://github.com/fribidi/fribidi</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>FriBidi/1.0.5-GCCcore-7.3.0</p></li>
<li><p>FriBidi/1.0.5-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###FSL
<strong>Description:</strong> FSL is a comprehensive library of analysis tools for FMRI, MRI and DTI brain imaging data.
<strong>Homepage:</strong> http://www.fmrib.ox.ac.uk/fsl/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>FSL/5.0.11-foss-2018b</p></li>
<li><p>FSL/5.0.11-foss-2018b-Python-3.6.6</p></li>
<li><p>FSL/6.0.1-foss-2019a-Python-3.7.2</p></li>
</ul>
</details>
</details>
<p>###GATK
<strong>Description:</strong> The Genome Analysis Toolkit or GATK is a software package developed at the Broad Institute
to analyse next-generation resequencing data. The toolkit offers a wide variety of tools,
with a primary focus on variant discovery and genotyping as well as strong emphasis on
data quality assurance. Its robust architecture, powerful processing engine and
high-performance computing features make it capable of taking on projects of any size.
<strong>Homepage:</strong> https://www.broadinstitute.org/gatk/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>GATK/4.1.0.0-foss-2018b-Python-3.6.6</p></li>
<li><p>GATK/4.1.0.0-intel-2018b-Python-3.6.6</p></li>
<li><p>GATK/4.1.2.0-GCCcore-8.2.0-Java-1.8</p></li>
<li><p>GATK/4.1.4.0-GCCcore-8.3.0-Java-1.8</p></li>
</ul>
</details>
</details>
<p>###Gaussian
<strong>Description:</strong> The Gaussian series of electronic structure programs provides a wide-ranging suite of the most advanced modeling capabilities available. You can use it to investigate the real-world chemical problems that interest you, in all of their complexity, even on modest computer hardware.
<strong>Homepage:</strong> http://gaussian.com</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Gaussian/g16_B.01</p></li>
<li><p>Gaussian/g16_C.01</p></li>
</ul>
</details>
</details>
<p>###gc
<strong>Description:</strong> The Boehm-Demers-Weiser conservative garbage collector can be used as a
garbage collecting replacement for C malloc or C++ new.</p>
<p><strong>Homepage:</strong> http://hboehm.info/gc/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>gc/7.6.0-GCCcore-6.4.0</p></li>
<li><p>gc/7.6.4-GCCcore-7.3.0</p></li>
<li><p>gc/7.6.10-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###GCC
<strong>Description:</strong> The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,
as well as libraries for these languages (libstdc++, libgcj,…).
<strong>Homepage:</strong> http://gcc.gnu.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>GCC/7.3.0-2.30</p></li>
<li><p>GCC/8.2.0-2.31.1</p></li>
<li><p>GCC/8.3.0</p></li>
</ul>
</details>
</details>
<p>###GCCcore
<strong>Description:</strong> The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,
as well as libraries for these languages (libstdc++, libgcj,…).
<strong>Homepage:</strong> http://gcc.gnu.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>GCCcore/6.4.0</p></li>
<li><p>GCCcore/7.3.0</p></li>
<li><p>GCCcore/8.2.0</p></li>
<li><p>GCCcore/8.3.0</p></li>
</ul>
</details>
</details>
<p>###gcccuda
<strong>Description:</strong> GNU Compiler Collection (GCC) based compiler toolchain, along with CUDA toolkit.
<strong>Homepage:</strong> (none)</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>gcccuda/2018b</p></li>
<li><p>gcccuda/2019a</p></li>
<li><p>gcccuda/2019b</p></li>
</ul>
</details>
</details>
<p>###GDAL
<strong>Description:</strong> GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style
Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model
to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for
data translation and processing.
<strong>Homepage:</strong> https://www.gdal.org</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>GDAL/2.2.3-foss-2018b-Python-3.6.6</p></li>
<li><p>GDAL/2.2.3-intel-2018b-Python-3.6.6</p></li>
<li><p>GDAL/2.4.0-foss-2019a-Python-3.7.2</p></li>
<li><p>GDAL/3.0.0-foss-2019a-Python-3.7.2</p></li>
<li><p>GDAL/3.0.0-intel-2019a-Python-3.7.2</p></li>
<li><p>GDAL/3.0.1-foss-2019a-Python-3.7.2</p></li>
<li><p>GDAL/3.0.2-intel-2019b-Python-3.7.4</p></li>
</ul>
</details>
</details>
<p>###GEOS
<strong>Description:</strong> GEOS (Geometry Engine
<strong>Homepage:</strong> https://trac.osgeo.org/geos</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>GEOS/3.6.2-foss-2018b-Python-3.6.6</p></li>
<li><p>GEOS/3.6.2-intel-2018b-Python-3.6.6</p></li>
<li><p>GEOS/3.7.2-foss-2019a-Python-3.7.2</p></li>
<li><p>GEOS/3.7.2-intel-2019a-Python-3.7.2</p></li>
<li><p>GEOS/3.8.0-iccifort-2019.5.281-Python-3.7.4</p></li>
</ul>
</details>
</details>
<p>###GetOrganelle
<strong>Description:</strong> The GetOrganelle toolkit assemblies organelle genome from genomic skimming data
<strong>Homepage:</strong> https://github.com/Kinggerm/GetOrganelle</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>GetOrganelle/1.6.2-foss-2019a-Python-3.7.2</p></li>
</ul>
</details>
</details>
<p>###gettext
<strong>Description:</strong> GNU ‘gettext’ is an important step for the GNU Translation Project, as it is an asset on which we may
build many other steps. This package offers to programmers, translators, and even users, a well integrated set of tools
and documentation
<strong>Homepage:</strong> https://www.gnu.org/software/gettext/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>gettext/0.19.8.1</p></li>
<li><p>gettext/0.19.8.1-GCCcore-7.3.0</p></li>
<li><p>gettext/0.19.8.1-GCCcore-8.2.0</p></li>
<li><p>gettext/0.20.1-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###Ghostscript
<strong>Description:</strong> Ghostscript is a versatile processor for PostScript data with the ability to render PostScript to
different targets. It used to be part of the cups printing stack, but is no longer used for that.
<strong>Homepage:</strong> https://ghostscript.com</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Ghostscript/9.27-GCCcore-8.2.0</p></li>
<li><p>Ghostscript/9.50-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###git
<strong>Description:</strong> Git is a free and open source distributed version control system designed
to handle everything from small to very large projects with speed and efficiency.
<strong>Homepage:</strong> https://git-scm.com/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>git/2.18.0-GCCcore-7.3.0</p></li>
<li><p>git/2.21.0-GCCcore-8.2.0</p></li>
<li><p>git/2.23.0-GCCcore-8.3.0</p></li>
<li><p>git/2.23.0-GCCcore-8.3.0-nodocs</p></li>
</ul>
</details>
</details>
<p>###GL2PS
<strong>Description:</strong> GL2PS: an OpenGL to PostScript printing library
<strong>Homepage:</strong> http://www.geuz.org/gl2ps/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>GL2PS/1.4.0-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###GLib
<strong>Description:</strong> GLib is one of the base libraries of the GTK+ project
<strong>Homepage:</strong> https://www.gtk.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>GLib/2.54.3-GCCcore-7.3.0</p></li>
<li><p>GLib/2.60.1-GCCcore-8.2.0</p></li>
<li><p>GLib/2.62.0-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###GLibmm
<strong>Description:</strong> C++ bindings for Glib
<strong>Homepage:</strong> http://www.gtk.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>GLibmm/2.49.7-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###GLM
<strong>Description:</strong> OpenGL Mathematics (GLM) is a header only C++ mathematics library for graphics software based on
the OpenGL Shading Language (GLSL) specifications.
<strong>Homepage:</strong> https://github.com/g-truc/glm</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>GLM/0.9.9.0-GCCcore-7.3.0</p></li>
<li><p>GLM/0.9.9.4-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###GMAP-GSNAP
<strong>Description:</strong> GMAP: A Genomic Mapping and Alignment Program for mRNA and EST Sequences
GSNAP: Genomic Short-read Nucleotide Alignment Program
<strong>Homepage:</strong> http://research-pub.gene.com/gmap/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>GMAP-GSNAP/2019-03-15-foss-2018b</p></li>
</ul>
</details>
</details>
<p>###GMP
<strong>Description:</strong> GMP is a free library for arbitrary precision arithmetic, operating on signed
integers, rational numbers, and floating point numbers.</p>
<p><strong>Homepage:</strong> https://gmplib.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>GMP/6.1.2-GCCcore-6.4.0</p></li>
<li><p>GMP/6.1.2-GCCcore-7.3.0</p></li>
<li><p>GMP/6.1.2-GCCcore-8.2.0</p></li>
<li><p>GMP/6.1.2-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###gmsh
<strong>Description:</strong> Gmsh is a 3D finite element grid generator with a build-in CAD engine and post-processor.
<strong>Homepage:</strong> http://geuz.org/gmsh</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>gmsh/4.2.2-foss-2018b-Python-3.6.6</p></li>
</ul>
</details>
</details>
<p>###gnuplot
<strong>Description:</strong> Portable interactive, function plotting utility
<strong>Homepage:</strong> http://gnuplot.sourceforge.net/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>gnuplot/5.2.5-foss-2018b</p></li>
<li><p>gnuplot/5.2.6-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###GObject-Introspection
<strong>Description:</strong> GObject introspection is a middleware layer between C libraries
(using GObject) and language bindings. The C library can be scanned at
compile time and generate a metadata file, in addition to the actual
native C library. Then at runtime, language bindings can read this
metadata and automatically provide bindings to call into the C library.
<strong>Homepage:</strong> https://wiki.gnome.org/GObjectIntrospection/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>GObject-Introspection/1.54.1-foss-2018b-Python-2.7.15</p></li>
<li><p>GObject-Introspection/1.60.1-GCCcore-8.2.0-Python-3.7.2</p></li>
</ul>
</details>
</details>
<p>###gompi
<strong>Description:</strong> GNU Compiler Collection (GCC) based compiler toolchain,
including OpenMPI for MPI support.
<strong>Homepage:</strong> (none)</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>gompi/2018b</p></li>
<li><p>gompi/2019a</p></li>
<li><p>gompi/2019b</p></li>
</ul>
</details>
</details>
<p>###gompic
<strong>Description:</strong> GNU Compiler Collection (GCC) based compiler toolchain along with CUDA toolkit,
including OpenMPI for MPI support with CUDA features enabled.
<strong>Homepage:</strong> (none)</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>gompic/2018b</p></li>
</ul>
</details>
</details>
<p>###gperf
<strong>Description:</strong> GNU gperf is a perfect hash function generator. For a given list of strings,
it produces a hash function and hash table, in form of C or C++ code, for
looking up a value depending on the input string. The hash function is
perfect, which means that the hash table has no collisions, and the hash
table lookup needs a single string comparison only.</p>
<p><strong>Homepage:</strong> https://www.gnu.org/software/gperf/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>gperf/3.1-GCCcore-7.3.0</p></li>
<li><p>gperf/3.1-GCCcore-8.2.0</p></li>
<li><p>gperf/3.1-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###GROMACS
<strong>Description:</strong> GROMACS is a versatile package to perform molecular dynamics,
i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles.</p>
<p>This is a CPU only build, containing both MPI and threadMPI builds.</p>
<p><strong>Homepage:</strong> http://www.gromacs.org</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>GROMACS/2018.1-foss-2018b</p></li>
<li><p>GROMACS/2018.4-fosscuda-2018b-PLUMED-2.5.0</p></li>
<li><p>GROMACS/2019-foss-2018b</p></li>
<li><p>GROMACS/2019.3-foss-2019a</p></li>
<li><p>GROMACS/2019.4-foss-2019b</p></li>
</ul>
</details>
</details>
<p>###GSL
<strong>Description:</strong> The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers.
The library provides a wide range of mathematical routines such as random number generators, special functions
and least-squares fitting.
<strong>Homepage:</strong> https://www.gnu.org/software/gsl/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>GSL/2.4-GCCcore-6.4.0</p></li>
<li><p>GSL/2.5-GCC-7.3.0-2.30</p></li>
<li><p>GSL/2.5-GCC-8.2.0-2.31.1</p></li>
<li><p>GSL/2.5-iccifort-2018.3.222-GCC-7.3.0-2.30</p></li>
<li><p>GSL/2.5-iccifort-2019.1.144-GCC-8.2.0-2.31.1</p></li>
<li><p>GSL/2.6-GCC-8.3.0</p></li>
<li><p>GSL/2.6-iccifort-2019.5.281</p></li>
</ul>
</details>
</details>
<p>###gtest
<strong>Description:</strong> Google’s framework for writing C++ tests on a variety of platforms
<strong>Homepage:</strong> https://github.com/google/googletest</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>gtest/1.8.1-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###Guile
<strong>Description:</strong> Guile is a programming language, designed to help programmers create flexible applications that
can be extended by users or other programmers with plug-ins, modules, or scripts.
<strong>Homepage:</strong> https://www.gnu.org/software/guile/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Guile/1.8.8-GCCcore-6.4.0</p></li>
<li><p>Guile/1.8.8-GCCcore-7.3.0</p></li>
<li><p>Guile/1.8.8-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###gzip
<strong>Description:</strong> gzip (GNU zip) is a popular data compression program as a replacement for compress
<strong>Homepage:</strong> http://www.gnu.org/software/gzip/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>gzip/1.10-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###h5py
<strong>Description:</strong> HDF5 for Python (h5py) is a general-purpose Python interface to the Hierarchical Data Format library,
version 5. HDF5 is a versatile, mature scientific software library designed for the fast, flexible storage of enormous
amounts of data.
<strong>Homepage:</strong> http://www.h5py.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>h5py/2.8.0-foss-2018b-Python-3.6.6</p></li>
<li><p>h5py/2.8.0-fosscuda-2018b-Python-3.6.6</p></li>
<li><p>h5py/2.8.0-intel-2018b-Python-3.6.6</p></li>
<li><p>h5py/2.9.0-foss-2019a</p></li>
<li><p>h5py/2.9.0-intel-2019a</p></li>
<li><p>h5py/2.10.0-foss-2019b-Python-3.7.4</p></li>
<li><p>h5py/2.10.0-intel-2019b-Python-3.7.4</p></li>
</ul>
</details>
</details>
<p>###HarfBuzz
<strong>Description:</strong> HarfBuzz is an OpenType text shaping engine.
<strong>Homepage:</strong> http://www.freedesktop.org/wiki/Software/HarfBuzz</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>HarfBuzz/2.2.0-foss-2018b</p></li>
<li><p>HarfBuzz/2.4.0-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###HDF5
<strong>Description:</strong> HDF5 is a data model, library, and file format for storing and managing data.
It supports an unlimited variety of datatypes, and is designed for flexible
and efficient I/O and for high volume and complex data.
<strong>Homepage:</strong> https://portal.hdfgroup.org/display/support</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>HDF5/1.10.1-intel-2018b</p></li>
<li><p>HDF5/1.10.2-foss-2018b</p></li>
<li><p>HDF5/1.10.2-fosscuda-2018b</p></li>
<li><p>HDF5/1.10.2-intel-2018b</p></li>
<li><p>HDF5/1.10.5-gompi-2019a</p></li>
<li><p>HDF5/1.10.5-gompi-2019b</p></li>
<li><p>HDF5/1.10.5-iimpi-2019a</p></li>
<li><p>HDF5/1.10.5-iimpi-2019b</p></li>
</ul>
</details>
</details>
<p>###help2man
<strong>Description:</strong> help2man produces simple manual pages from the ‘–help’ and ‘–version’ output of other commands.
<strong>Homepage:</strong> https://www.gnu.org/software/help2man/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>help2man/1.47.4</p></li>
<li><p>help2man/1.47.4-GCCcore-6.4.0</p></li>
<li><p>help2man/1.47.4-GCCcore-7.3.0</p></li>
<li><p>help2man/1.47.7-GCCcore-8.2.0</p></li>
<li><p>help2man/1.47.8-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###HipSTR
<strong>Description:</strong> Haplotype inference and phasing for Short Tandem Repeats
<strong>Homepage:</strong> https://github.com/tfwillems/HipSTR</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>HipSTR/0.6.2-foss-2018b-Python-3.6.6</p></li>
</ul>
</details>
</details>
<p>###HISAT2
<strong>Description:</strong> HISAT2 is a fast and sensitive alignment program for mapping next-generation sequencing reads
(both DNA and RNA) against the general human population (as well as against a single reference genome).
<strong>Homepage:</strong> https://ccb.jhu.edu/software/hisat2/index.shtml</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>HISAT2/2.1.0-foss-2018b</p></li>
</ul>
</details>
</details>
<p>###HMMER
<strong>Description:</strong> HMMER is used for searching sequence databases for homologs
of protein sequences, and for making protein sequence alignments. It
implements methods using probabilistic models called profile hidden Markov
models (profile HMMs).  Compared to BLAST, FASTA, and other sequence
alignment and database search tools based on older scoring methodology,
HMMER aims to be significantly more accurate and more able to detect remote
homologs because of the strength of its underlying mathematical models. In the
past, this strength came at significant computational expense, but in the new
HMMER3 project, HMMER is now essentially as fast as BLAST.
<strong>Homepage:</strong> http://hmmer.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>HMMER/2.3.2-GCC-7.3.0-2.30</p></li>
<li><p>HMMER/3.2.1-GCC-8.2.0-2.31.1</p></li>
<li><p>HMMER/3.2.1-foss-2018b</p></li>
<li><p>HMMER/3.2.1-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###htop
<strong>Description:</strong> An interactive process viewer for Unix
<strong>Homepage:</strong> http://hisham.hm/htop/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>htop/2.2.0</p></li>
</ul>
</details>
</details>
<p>###HTSeq
<strong>Description:</strong> HTSeq is a Python library to facilitate processing and analysis
of data from high-throughput sequencing (HTS) experiments.
<strong>Homepage:</strong> https://github.com/simon-anders/htseq</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>HTSeq/0.11.2-foss-2019a-Python-3.7.2</p></li>
</ul>
</details>
</details>
<p>###HTSlib
<strong>Description:</strong> A C library for reading/writing high-throughput sequencing data.
This package includes the utilities bgzip and tabix
<strong>Homepage:</strong> http://www.htslib.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>HTSlib/1.9-GCC-7.3.0-2.30</p></li>
<li><p>HTSlib/1.9-GCC-8.2.0-2.31.1</p></li>
<li><p>HTSlib/1.9-foss-2018b</p></li>
<li><p>HTSlib/1.9-iccifort-2019.1.144-GCC-8.2.0-2.31.1</p></li>
<li><p>HTSlib/1.9-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###hwloc
<strong>Description:</strong> The Portable Hardware Locality (hwloc) software package provides a portable
abstraction (across OS, versions, architectures, …) of the hierarchical
topology of modern architectures, including NUMA memory nodes, sockets, shared
caches, cores and simultaneous multithreading. It also gathers various system
attributes such as cache and memory information as well as the locality of I/O
devices such as network interfaces, InfiniBand HCAs or GPUs. It primarily
aims at helping applications with gathering information about modern computing
hardware so as to exploit it accordingly and efficiently.</p>
<p><strong>Homepage:</strong> https://www.open-mpi.org/projects/hwloc/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>hwloc/1.11.10-GCCcore-7.3.0</p></li>
<li><p>hwloc/1.11.12-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###hypothesis
<strong>Description:</strong> Hypothesis is an advanced testing library for Python. It lets you write tests which are parametrized
by a source of examples, and then generates simple and comprehensible examples that make your tests fail. This lets
you find more bugs in your code with less work.
<strong>Homepage:</strong> https://github.com/HypothesisWorks/hypothesis</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>hypothesis/4.5.0-fosscuda-2018b-Python-3.6.6</p></li>
</ul>
</details>
</details>
<p>###Hypre
<strong>Description:</strong> Hypre is a library for solving large, sparse linear systems of equations on massively
parallel computers. The problems of interest arise in the simulation codes being developed at LLNL
and elsewhere to study physical phenomena in the defense, environmental, energy, and biological sciences.
<strong>Homepage:</strong> https://computation.llnl.gov/projects/hypre-scalable-linear-solvers-multigrid-methods</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Hypre/2.15.0-foss-2018b</p></li>
<li><p>Hypre/2.15.0-intel-2018b</p></li>
<li><p>Hypre/2.15.1-foss-2019a</p></li>
<li><p>Hypre/2.15.1-intel-2019a</p></li>
</ul>
</details>
</details>
<p>###icc
<strong>Description:</strong> Intel C and C++ compilers
<strong>Homepage:</strong> http://software.intel.com/en-us/intel-compilers/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>icc/2018.1.163-GCC-6.4.0-2.28</p></li>
<li><p>icc/2018.3.222-GCC-7.3.0-2.30</p></li>
<li><p>icc/2019.1.144-GCC-8.2.0-2.31.1</p></li>
</ul>
</details>
</details>
<p>###iccifort
<strong>Description:</strong> Intel C, C++ &amp; Fortran compilers
<strong>Homepage:</strong> http://software.intel.com/en-us/intel-compilers/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>iccifort/2018.1.163-GCC-6.4.0-2.28</p></li>
<li><p>iccifort/2018.3.222-GCC-7.3.0-2.30</p></li>
<li><p>iccifort/2019.1.144-GCC-8.2.0-2.31.1</p></li>
<li><p>iccifort/2019.5.281</p></li>
</ul>
</details>
</details>
<p>###iccifortcuda
<strong>Description:</strong> Intel C, C++ &amp; Fortran compilers with CUDA toolkit
<strong>Homepage:</strong> (none)</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>iccifortcuda/2019a</p></li>
<li><p>iccifortcuda/2019b</p></li>
</ul>
</details>
</details>
<p>###ICU
<strong>Description:</strong> ICU is a mature, widely used set of C/C++ and Java libraries providing Unicode and Globalization
support for software applications.
<strong>Homepage:</strong> http://site.icu-project.org/home</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ICU/61.1-GCCcore-7.3.0</p></li>
<li><p>ICU/64.2-GCCcore-8.2.0</p></li>
<li><p>ICU/64.2-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###ifort
<strong>Description:</strong> Intel Fortran compiler
<strong>Homepage:</strong> http://software.intel.com/en-us/intel-compilers/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ifort/2018.1.163-GCC-6.4.0-2.28</p></li>
<li><p>ifort/2018.3.222-GCC-7.3.0-2.30</p></li>
<li><p>ifort/2019.1.144-GCC-8.2.0-2.31.1</p></li>
</ul>
</details>
</details>
<p>###iimpi
<strong>Description:</strong> Intel C/C++ and Fortran compilers, alongside Intel MPI.
<strong>Homepage:</strong> http://software.intel.com/en-us/intel-cluster-toolkit-compiler/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>iimpi/2018a</p></li>
<li><p>iimpi/2018b</p></li>
<li><p>iimpi/2019a</p></li>
<li><p>iimpi/2019b</p></li>
</ul>
</details>
</details>
<p>###iimpic
<strong>Description:</strong> Intel C/C++ and Fortran compilers, alongside Intel MPI and CUDA.
<strong>Homepage:</strong> (none)</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>iimpic/2019a</p></li>
<li><p>iimpic/2019b</p></li>
</ul>
</details>
</details>
<p>###ImageMagick
<strong>Description:</strong> ImageMagick is a software suite to create, edit, compose, or convert bitmap images
<strong>Homepage:</strong> https://www.imagemagick.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ImageMagick/7.0.8-46-GCCcore-8.2.0</p></li>
<li><p>ImageMagick/7.0.9-5-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###imkl
<strong>Description:</strong> Intel Math Kernel Library is a library of highly optimized,
extensively threaded math routines for science, engineering, and financial
applications that require maximum performance. Core math functions include
BLAS, LAPACK, ScaLAPACK, Sparse Solvers, Fast Fourier Transforms, Vector Math, and more.
<strong>Homepage:</strong> https://software.intel.com/en-us/intel-mkl/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>imkl/2018.1.163-iimpi-2018a</p></li>
<li><p>imkl/2018.3.222-iimpi-2018b</p></li>
<li><p>imkl/2018.3.222-iompi-2018b</p></li>
<li><p>imkl/2019.1.144-iimpi-2019a</p></li>
<li><p>imkl/2019.1.144-iimpic-2019a</p></li>
<li><p>imkl/2019.5.281-iimpi-2019b</p></li>
<li><p>imkl/2019.5.281-iimpic-2019b</p></li>
</ul>
</details>
</details>
<p>###impi
<strong>Description:</strong> Intel MPI Library, compatible with MPICH ABI
<strong>Homepage:</strong> https://software.intel.com/en-us/intel-mpi-library/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>impi/2018.1.163-iccifort-2018.1.163-GCC-6.4.0-2.28</p></li>
<li><p>impi/2018.3.222-iccifort-2018.3.222-GCC-7.3.0-2.30</p></li>
<li><p>impi/2018.4.274-iccifort-2019.1.144-GCC-8.2.0-2.31.1</p></li>
<li><p>impi/2018.4.274-iccifortcuda-2019a</p></li>
<li><p>impi/2018.5.288-iccifort-2019.5.281</p></li>
<li><p>impi/2018.5.288-iccifortcuda-2019b</p></li>
</ul>
</details>
</details>
<p>###intel
<strong>Description:</strong> Compiler toolchain including Intel compilers, Intel MPI and Intel Math Kernel Library (MKL).
<strong>Homepage:</strong> https://easybuild.readthedocs.io/en/master/Common-toolchains.html#intel-toolchain</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>intel/2018a</p></li>
<li><p>intel/2018b</p></li>
<li><p>intel/2019a</p></li>
<li><p>intel/2019b</p></li>
</ul>
</details>
</details>
<p>###intelcuda
<strong>Description:</strong> Intel Cluster Toolkit Compiler Edition provides Intel C/C++ and Fortran compilers,
Intel MPI &amp; Intel MKL, with CUDA toolkit
<strong>Homepage:</strong> (none)</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>intelcuda/2019a</p></li>
<li><p>intelcuda/2019b</p></li>
</ul>
</details>
</details>
<p>###InterProScan
<strong>Description:</strong> InterProScan is a sequence analysis application (nucleotide and protein sequences) that combines
different protein signature recognition methods into one resource.
<strong>Homepage:</strong> http://www.ebi.ac.uk/interpro/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>InterProScan/5.34-73.0-intel-2019a</p></li>
</ul>
</details>
</details>
<p>###intltool
<strong>Description:</strong> intltool is a set of tools to centralize translation of
many different file formats using GNU gettext-compatible PO files.
<strong>Homepage:</strong> http://freedesktop.org/wiki/Software/intltool/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>intltool/0.51.0-GCCcore-7.3.0-Perl-5.28.0</p></li>
<li><p>intltool/0.51.0-GCCcore-8.2.0</p></li>
<li><p>intltool/0.51.0-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###iomkl
<strong>Description:</strong> Intel Cluster Toolchain Compiler Edition provides Intel C/C++ and Fortran compilers, Intel MKL &amp;
OpenMPI.
<strong>Homepage:</strong> http://software.intel.com/en-us/intel-cluster-toolkit-compiler/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>iomkl/2018b</p></li>
</ul>
</details>
</details>
<p>###iompi
<strong>Description:</strong> Intel C/C++ and Fortran compilers, alongside Open MPI.
<strong>Homepage:</strong> http://software.intel.com/en-us/intel-cluster-toolkit-compiler/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>iompi/2018b</p></li>
</ul>
</details>
</details>
<p>###ipyparallel
<strong>Description:</strong> ipyparallel is a Python package and collection
of CLI scripts for controlling clusters for Jupyter
<strong>Homepage:</strong> https://ipyparallel.readthedocs.io</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ipyparallel/6.2.4-foss-2018b-Python-3.6.6</p></li>
</ul>
</details>
</details>
<p>###ipyrad
<strong>Description:</strong> ipyrad is an interactive toolkit for assembly and analysis of restriction-site associated genomic
data sets (e.g., RAD, ddRAD, GBS) for population genetic and phylogenetic studies.
<strong>Homepage:</strong> https://ipyrad.readthedocs.io</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ipyrad/0.9.18-foss-2018b-Python-3.6.6</p></li>
</ul>
</details>
</details>
<p>###IPython
<strong>Description:</strong> IPython provides a rich architecture for interactive computing with:
Powerful interactive shells (terminal and Qt-based).
A browser-based notebook with support for code, text, mathematical expressions, inline plots and other rich media.
Support for interactive data visualization and use of GUI toolkits.
Flexible, embeddable interpreters to load into your own projects.
Easy to use, high performance tools for parallel computing.
<strong>Homepage:</strong> http://ipython.org/index.html</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>IPython/5.8.0-foss-2018b-Python-2.7.15</p></li>
<li><p>IPython/7.2.0-foss-2018b-Python-3.6.6</p></li>
</ul>
</details>
</details>
<p>###IQ-TREE
<strong>Description:</strong> Efficient phylogenomic software by maximum likelihood
<strong>Homepage:</strong> http://www.iqtree.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>IQ-TREE/1.6.8-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###ITK
<strong>Description:</strong> Insight Segmentation and Registration Toolkit (ITK) provides
an extensive suite of software tools for registering and segmenting
multidimensional imaging data.
<strong>Homepage:</strong> https://itk.org</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ITK/4.13.1-foss-2018b-Python-3.6.6</p></li>
</ul>
</details>
</details>
<p>###JasPer
<strong>Description:</strong> The JasPer Project is an open-source initiative to provide a free
software-based reference implementation of the codec specified in
the JPEG-2000 Part-1 standard.</p>
<p><strong>Homepage:</strong> https://www.ece.uvic.ca/~frodo/jasper/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>JasPer/2.0.14-GCCcore-7.3.0</p></li>
<li><p>JasPer/2.0.14-GCCcore-8.2.0</p></li>
<li><p>JasPer/2.0.14-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###Java
<strong>Description:</strong> Java Platform, Standard Edition (Java SE) lets you develop and deploy
Java applications on desktops and servers.
<strong>Homepage:</strong> http://openjdk.java.net</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Java/1.7.0_80</p></li>
<li><p>Java/1.8.0_212</p></li>
<li><p>Java/11.0.2</p></li>
</ul>
</details>
</details>
<p>###Jellyfish
<strong>Description:</strong> Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA.
<strong>Homepage:</strong> http://www.genome.umd.edu/jellyfish.html</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Jellyfish/2.2.10-foss-2018b</p></li>
<li><p>Jellyfish/2.3.0-GCC-8.2.0-2.31.1</p></li>
</ul>
</details>
</details>
<p>###jemalloc
<strong>Description:</strong> jemalloc is a general purpose malloc(3) implementation that emphasizes fragmentation avoidance and
scalable concurrency support.
<strong>Homepage:</strong> http://jemalloc.net</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>jemalloc/5.1.0-GCCcore-7.3.0</p></li>
<li><p>jemalloc/5.2.0-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###Julia
<strong>Description:</strong> Julia as an open source high performance programming language.
<strong>Homepage:</strong> https://julialang.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Julia/1.1.1</p></li>
</ul>
</details>
</details>
<p>###Keras
<strong>Description:</strong> Keras is a minimalist, highly modular neural networks library, written in Python and
capable of running on top of either TensorFlow or Theano.
<strong>Homepage:</strong> https://keras.io/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Keras/2.2.2-fosscuda-2018b-Python-3.6.6</p></li>
</ul>
</details>
</details>
<p>###khmer
<strong>Description:</strong> In-memory nucleotide sequence k-mer counting, filtering, graph traversal and more
<strong>Homepage:</strong> https://github.com/ged-lab/khmer/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>khmer/2.1.1-intel-2018b-Python-2.7.15</p></li>
</ul>
</details>
</details>
<p>###Kraken2
<strong>Description:</strong> Kraken is a system for assigning taxonomic labels to short DNA sequences,
usually obtained through metagenomic studies. Previous attempts by other
bioinformatics software to accomplish this task have often used sequence
alignment or machine learning techniques that were quite slow, leading to
the development of less sensitive but much faster abundance estimation
programs. Kraken aims to achieve high sensitivity and high speed by
utilizing exact alignments of k-mers and a novel classification algorithm.
<strong>Homepage:</strong> http://www.ccb.jhu.edu/software/kraken2/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Kraken2/2.0.7-beta-foss-2018b-Perl-5.28.0</p></li>
<li><p>Kraken2/2.0.8-beta-foss-2018b-Perl-5.28.0</p></li>
</ul>
</details>
</details>
<p>###L_RNA_scaffolder
<strong>Description:</strong> L_RNA_scaffolder is a genome scaffolding tool with long trancriptome reads
<strong>Homepage:</strong> https://github.com/CAFS-bioinformatics/L_RNA_scaffolder</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>L_RNA_scaffolder/190530</p></li>
</ul>
</details>
</details>
<p>###LAMMPS
<strong>Description:</strong> LAMMPS (Large-scale Atomic/Molecular Massively Parallel Simulator) is a classical molecular dynamics simulation code designed to run efficiently on parallel computers.
<strong>Homepage:</strong> http://lammps.sandia.gov/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>LAMMPS/7Aug19-foss-2018b-Python-2.7.15</p></li>
</ul>
</details>
</details>
<p>###libcerf
<strong>Description:</strong> libcerf is a self-contained numeric library that provides an efficient and
accurate implementation of complex error functions, along with Dawson,
Faddeeva, and Voigt functions.</p>
<p><strong>Homepage:</strong> http://apps.jcns.fz-juelich.de/doku/sc/libcerf</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>libcerf/1.7-GCCcore-7.3.0</p></li>
<li><p>libcerf/1.11-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###libdap
<strong>Description:</strong> A C++ SDK which contains an implementation of DAP 2.0
and the development versions of DAP3, up to 3.4.
This includes both Client- and Server-side support classes.
<strong>Homepage:</strong> http://opendap.org/download/libdap</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>libdap/3.20.3-GCCcore-7.3.0</p></li>
</ul>
</details>
</details>
<p>###libdrm
<strong>Description:</strong> Direct Rendering Manager runtime library.
<strong>Homepage:</strong> http://dri.freedesktop.org</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>libdrm/2.4.92-GCCcore-7.3.0</p></li>
<li><p>libdrm/2.4.97-GCCcore-8.2.0</p></li>
<li><p>libdrm/2.4.99-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###libevent
<strong>Description:</strong> The libevent API provides a mechanism to execute a callback function when
a specific event occurs on a file descriptor or after a timeout has been
reached.  Furthermore, libevent also support callbacks due to signals or
regular timeouts.</p>
<p><strong>Homepage:</strong> https://libevent.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>libevent/2.1.11-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###libffi
<strong>Description:</strong> The libffi library provides a portable, high level programming interface to
various calling conventions. This allows a programmer to call any function
specified by a call interface description at run-time.</p>
<p><strong>Homepage:</strong> https://sourceware.org/libffi/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>libffi/3.2.1-GCCcore-6.4.0</p></li>
<li><p>libffi/3.2.1-GCCcore-7.3.0</p></li>
<li><p>libffi/3.2.1-GCCcore-8.2.0</p></li>
<li><p>libffi/3.2.1-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###libgd
<strong>Description:</strong> GD is an open source code library for the dynamic creation of images by programmers.
<strong>Homepage:</strong> https://libgd.github.io/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>libgd/2.2.5-GCCcore-7.3.0</p></li>
<li><p>libgd/2.2.5-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###libgeotiff
<strong>Description:</strong> Library for reading and writing coordinate system information from/to GeoTIFF files
<strong>Homepage:</strong> https://directory.fsf.org/wiki/Libgeotiff</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>libgeotiff/1.4.2-foss-2018b</p></li>
<li><p>libgeotiff/1.4.2-intel-2018b</p></li>
<li><p>libgeotiff/1.5.1-GCCcore-8.2.0</p></li>
<li><p>libgeotiff/1.5.1-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###libGLU
<strong>Description:</strong> The OpenGL Utility Library (GLU) is a computer graphics library for OpenGL.
<strong>Homepage:</strong> ftp://ftp.freedesktop.org/pub/mesa/glu/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>libGLU/9.0.0-GCCcore-8.2.0</p></li>
<li><p>libGLU/9.0.0-foss-2018b</p></li>
<li><p>libGLU/9.0.0-intel-2018b</p></li>
<li><p>libGLU/9.0.1-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###libgpuarray
<strong>Description:</strong> Library to manipulate tensors on the GPU.
<strong>Homepage:</strong> http://deeplearning.net/software/libgpuarray/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>libgpuarray/0.7.6-fosscuda-2018b-Python-3.6.6</p></li>
</ul>
</details>
</details>
<p>###libgtextutils
<strong>Description:</strong> ligtextutils is a dependency of fastx-toolkit and is provided via the same upstream
<strong>Homepage:</strong> http://hannonlab.cshl.edu/fastx_toolkit/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>libgtextutils/0.7-GCCcore-7.3.0</p></li>
</ul>
</details>
</details>
<p>###libharu
<strong>Description:</strong> libHaru is a free, cross platform, open source library for generating PDF files.
<strong>Homepage:</strong> http://libharu.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>libharu/2.3.0-GCCcore-7.3.0</p></li>
<li><p>libharu/2.3.0-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###Libint
<strong>Description:</strong> Libint library is used to evaluate the traditional (electron repulsion) and certain novel two-body
matrix elements (integrals) over Cartesian Gaussian functions used in modern atomic and molecular theory.
<strong>Homepage:</strong> https://sourceforge.net/p/libint/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Libint/1.1.6-GCC-8.2.0-2.31.1</p></li>
<li><p>Libint/1.1.6-iccifort-2019.1.144-GCC-8.2.0-2.31.1</p></li>
<li><p>Libint/1.1.6-intel-2018a</p></li>
<li><p>Libint/1.1.6-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###libjpeg-turbo
<strong>Description:</strong> libjpeg-turbo is a fork of the original IJG libjpeg which uses SIMD to
accelerate baseline JPEG compression and decompression. libjpeg is a library
that implements JPEG image encoding, decoding and transcoding.</p>
<p><strong>Homepage:</strong> https://sourceforge.net/projects/libjpeg-turbo/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>libjpeg-turbo/2.0.0-GCCcore-7.3.0</p></li>
<li><p>libjpeg-turbo/2.0.2-GCCcore-8.2.0</p></li>
<li><p>libjpeg-turbo/2.0.3-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###libmatheval
<strong>Description:</strong> GNU libmatheval is a library (callable from C and Fortran) to parse
and evaluate symbolic expressions input as text.
<strong>Homepage:</strong> http://www.gnu.org/software/libmatheval/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>libmatheval/1.1.11-GCCcore-6.4.0</p></li>
<li><p>libmatheval/1.1.11-GCCcore-7.3.0</p></li>
<li><p>libmatheval/1.1.11-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###libpciaccess
<strong>Description:</strong> Generic PCI access library.
<strong>Homepage:</strong> http://cgit.freedesktop.org/xorg/lib/libpciaccess/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>libpciaccess/0.14-GCCcore-7.3.0</p></li>
<li><p>libpciaccess/0.14-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###libpng
<strong>Description:</strong> libpng is the official PNG reference library
<strong>Homepage:</strong> http://www.libpng.org/pub/png/libpng.html</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>libpng/1.2.58</p></li>
<li><p>libpng/1.6.34-GCCcore-7.3.0</p></li>
<li><p>libpng/1.6.36-GCCcore-8.2.0</p></li>
<li><p>libpng/1.6.37-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###libreadline
<strong>Description:</strong> The GNU Readline library provides a set of functions for use by applications
that allow users to edit command lines as they are typed in. Both Emacs and
vi editing modes are available. The Readline library includes additional
functions to maintain a list of previously-entered command lines, to recall
and perhaps reedit those lines, and perform csh-like history expansion on
previous commands.</p>
<p><strong>Homepage:</strong> https://tiswww.case.edu/php/chet/readline/rltop.html</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>libreadline/7.0-GCCcore-7.3.0</p></li>
<li><p>libreadline/8.0-GCCcore-8.2.0</p></li>
<li><p>libreadline/8.0-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###libsigc++
<strong>Description:</strong> The libsigc++ package implements a typesafe callback system for standard C++.
<strong>Homepage:</strong> https://libsigcplusplus.github.io/libsigcplusplus/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>libsigc++/2.10.2-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###libsndfile
<strong>Description:</strong> Libsndfile is a C library for reading and writing files containing sampled sound
(such as MS Windows WAV and the Apple/SGI AIFF format) through one standard library interface.
<strong>Homepage:</strong> http://www.mega-nerd.com/libsndfile</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>libsndfile/1.0.28-GCCcore-7.3.0</p></li>
<li><p>libsndfile/1.0.28-GCCcore-8.2.0</p></li>
<li><p>libsndfile/1.0.28-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###libsodium
<strong>Description:</strong> Sodium is a modern, easy-to-use software library for encryption, decryption,
signatures, password hashing and more.</p>
<p><strong>Homepage:</strong> http://doc.libsodium.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>libsodium/1.0.16-GCCcore-7.3.0</p></li>
</ul>
</details>
</details>
<p>###LibTIFF
<strong>Description:</strong> tiff: Library and tools for reading and writing TIFF data files
<strong>Homepage:</strong> http://www.remotesensing.org/libtiff/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>LibTIFF/4.0.9-GCCcore-7.3.0</p></li>
<li><p>LibTIFF/4.0.9-intel-2018b</p></li>
<li><p>LibTIFF/4.0.10-GCCcore-8.2.0</p></li>
<li><p>LibTIFF/4.0.10-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###libtool
<strong>Description:</strong> GNU libtool is a generic library support script. Libtool hides the complexity
of using shared libraries behind a consistent, portable interface.</p>
<p><strong>Homepage:</strong> http://www.gnu.org/software/libtool</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>libtool/2.4.6-GCCcore-6.4.0</p></li>
<li><p>libtool/2.4.6-GCCcore-7.3.0</p></li>
<li><p>libtool/2.4.6-GCCcore-8.2.0</p></li>
<li><p>libtool/2.4.6-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###libunistring
<strong>Description:</strong> This library provides functions for manipulating Unicode strings and for
manipulating C strings according to the Unicode standard.</p>
<p><strong>Homepage:</strong> http://www.gnu.org/software/libunistring/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>libunistring/0.9.7-GCCcore-6.4.0</p></li>
<li><p>libunistring/0.9.10-GCCcore-7.3.0</p></li>
<li><p>libunistring/0.9.10-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###libunwind
<strong>Description:</strong> The primary goal of libunwind is to define a portable and efficient C programming interface
(API) to determine the call-chain of a program. The API additionally provides the means to manipulate the
preserved (callee-saved) state of each call-frame and to resume execution at any point in the call-chain
(non-local goto). The API supports both local (same-process) and remote (across-process) operation.
As such, the API is useful in a number of applications
<strong>Homepage:</strong> https://www.nongnu.org/libunwind/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>libunwind/1.2.1-GCCcore-7.3.0</p></li>
<li><p>libunwind/1.3.1-GCCcore-8.2.0</p></li>
<li><p>libunwind/1.3.1-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###libxc
<strong>Description:</strong> Libxc is a library of exchange-correlation functionals for density-functional theory.
The aim is to provide a portable, well tested and reliable set of exchange and correlation functionals.
<strong>Homepage:</strong> http://www.tddft.org/programs/octopus/wiki/index.php/Libxc</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>libxc/4.2.3-intel-2018a</p></li>
<li><p>libxc/4.2.3-intel-2018b</p></li>
<li><p>libxc/4.3.4-GCC-8.2.0-2.31.1</p></li>
<li><p>libxc/4.3.4-iccifort-2019.1.144-GCC-8.2.0-2.31.1</p></li>
</ul>
</details>
</details>
<p>###libxml++
<strong>Description:</strong> libxml++ is a C++ wrapper for the libxml XML parser library.
<strong>Homepage:</strong> http://libxmlplusplus.sourceforge.net</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>libxml++/2.40.1-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###libxml2
<strong>Description:</strong> Libxml2 is the XML C parser and toolchain developed for the Gnome project
(but usable outside of the Gnome platform).</p>
<p><strong>Homepage:</strong> http://xmlsoft.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>libxml2/2.9.8-GCCcore-7.3.0</p></li>
<li><p>libxml2/2.9.8-GCCcore-8.2.0</p></li>
<li><p>libxml2/2.9.9-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###libxslt
<strong>Description:</strong> Libxslt is the XSLT C library developed for the GNOME project
(but usable outside of the Gnome platform).
<strong>Homepage:</strong> http://xmlsoft.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>libxslt/1.1.32-GCCcore-7.3.0</p></li>
<li><p>libxslt/1.1.33-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###libxsmm
<strong>Description:</strong> LIBXSMM is a library for small dense and small sparse matrix-matrix multiplications
targeting Intel Architecture (x86).
<strong>Homepage:</strong> https://github.com/hfp/libxsmm</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>libxsmm/1.8.3-intel-2018a</p></li>
<li><p>libxsmm/1.10-GCC-8.2.0-2.31.1</p></li>
<li><p>libxsmm/1.10-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###libyaml
<strong>Description:</strong> LibYAML is a YAML parser and emitter written in C.
<strong>Homepage:</strong> http://pyyaml.org/wiki/LibYAML</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>libyaml/0.2.1-GCCcore-7.3.0</p></li>
<li><p>libyaml/0.2.2-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###LINKS
<strong>Description:</strong> LINKS is a genomics application for scaffolding genome assemblies with long reads
<strong>Homepage:</strong> https://github.com/bcgsc/LINKS</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>LINKS/1.8.7-GCC-8.2.0-2.31.1-Perl-5.28.1</p></li>
</ul>
</details>
</details>
<p>###LittleCMS
<strong>Description:</strong> Little CMS intends to be an OPEN SOURCE small-footprint color management engine,
with special focus on accuracy and performance.
<strong>Homepage:</strong> http://www.littlecms.com/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>LittleCMS/2.9-GCCcore-8.2.0</p></li>
<li><p>LittleCMS/2.9-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###LLVM
<strong>Description:</strong> The LLVM Core libraries provide a modern source- and target-independent
optimizer, along with code generation support for many popular CPUs
(as well as some less common ones!) These libraries are built around a well
specified code representation known as the LLVM intermediate representation
(“LLVM IR”). The LLVM Core libraries are well documented, and it is
particularly easy to invent your own language (or port an existing compiler)
to use LLVM as an optimizer and code generator.
<strong>Homepage:</strong> http://llvm.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>LLVM/6.0.0-GCCcore-7.3.0</p></li>
<li><p>LLVM/6.0.1-GCCcore-7.3.0</p></li>
<li><p>LLVM/7.0.1-GCCcore-8.2.0</p></li>
<li><p>LLVM/9.0.0-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###LMDB
<strong>Description:</strong> LMDB is a fast, memory-efficient database. With memory-mapped files, it has the read performance
of a pure in-memory database while retaining the persistence of standard disk-based databases.
<strong>Homepage:</strong> https://symas.com/lmdb</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>LMDB/0.9.22-GCCcore-7.3.0</p></li>
<li><p>LMDB/0.9.23-GCCcore-8.2.0</p></li>
<li><p>LMDB/0.9.24-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###Loter
<strong>Description:</strong> A Python (and soon R) package for local ancestry inference and haplotype phasing.
<strong>Homepage:</strong> https://github.com/bcm-uga/Loter</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Loter/191204-foss-2019a-Python-2.7.15</p></li>
</ul>
</details>
</details>
<p>###lpsolve
<strong>Description:</strong> Mixed Integer Linear Programming (MILP) solver
<strong>Homepage:</strong> https://sourceforge.net/projects/lpsolve/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>lpsolve/5.5.2.5-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###lxml
<strong>Description:</strong> The lxml XML toolkit is a Pythonic binding for the C libraries libxml2 and libxslt.
<strong>Homepage:</strong> http://lxml.de/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>lxml/4.2.5-intel-2018b-Python-3.6.6</p></li>
<li><p>lxml/4.3.3-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###lz4
<strong>Description:</strong> LZ4 is lossless compression algorithm, providing compression speed at 400 MB/s per core.
It features an extremely fast decoder, with speed in multiple GB/s per core.
<strong>Homepage:</strong> https://lz4.github.io/lz4/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>lz4/1.9.1-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###LZO
<strong>Description:</strong> Portable lossless data compression library
<strong>Homepage:</strong> http://www.oberhumer.com/opensource/lzo/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>LZO/2.10-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###M4
<strong>Description:</strong> GNU M4 is an implementation of the traditional Unix macro processor. It is
mostly SVR4 compatible although it has some extensions (for example, handling
more than 9 positional parameters to macros). GNU M4 also has built-in
functions for including files, running shell commands, doing arithmetic, etc.</p>
<p><strong>Homepage:</strong> http://www.gnu.org/software/m4/m4.html</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>M4/1.4.17</p></li>
<li><p>M4/1.4.18</p></li>
<li><p>M4/1.4.18-GCCcore-6.4.0</p></li>
<li><p>M4/1.4.18-GCCcore-7.3.0</p></li>
<li><p>M4/1.4.18-GCCcore-8.2.0</p></li>
<li><p>M4/1.4.18-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###MACS2
<strong>Description:</strong> Model Based Analysis for ChIP-Seq data
<strong>Homepage:</strong> https://github.com/taoliu/MACS/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>MACS2/2.1.2.1-foss-2018b-Python-2.7.15</p></li>
<li><p>MACS2/2.1.2.1-intel-2018b-Python-2.7.15</p></li>
</ul>
</details>
</details>
<p>###MAFFT
<strong>Description:</strong> MAFFT is a multiple sequence alignment program
for unix-like operating systems.  It offers a range of multiple
alignment methods, L-INS-i (accurate; for alignment of &lt;∼200 sequences),
FFT-NS-2 (fast; for alignment of &lt;∼10,000 sequences), etc.
<strong>Homepage:</strong> https://mafft.cbrc.jp/alignment/software/source.html</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>MAFFT/7.427-intel-2018b-with-extensions</p></li>
<li><p>MAFFT/7.429-GCC-8.2.0-2.31.1-with-extensions</p></li>
</ul>
</details>
</details>
<p>###magma
<strong>Description:</strong> The MAGMA project aims to develop a dense linear algebra library similar to
LAPACK but for heterogeneous/hybrid architectures, starting with current Multicore+GPU systems.
<strong>Homepage:</strong> http://icl.cs.utk.edu/magma/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>magma/2.5.0-fosscuda-2018b</p></li>
</ul>
</details>
</details>
<p>###Maker
<strong>Description:</strong> A portable and easily configurable genome annotation pipeline.
<strong>Homepage:</strong> https://www.yandell-lab.org/software/maker.html</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Maker/2.31.10-foss-2019a</p></li>
<li><p>Maker/3.01.02-beta-foss-2019a</p></li>
</ul>
</details>
</details>
<p>###Mako
<strong>Description:</strong> A super-fast templating language that borrows the best ideas from the existing templating languages
<strong>Homepage:</strong> http://www.makotemplates.org</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Mako/1.0.7-foss-2018b-Python-2.7.15</p></li>
<li><p>Mako/1.0.7-fosscuda-2018b-Python-3.6.6</p></li>
<li><p>Mako/1.0.7-intel-2018b-Python-2.7.15</p></li>
<li><p>Mako/1.0.8-GCCcore-8.2.0</p></li>
<li><p>Mako/1.1.0-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###mapDamage
<strong>Description:</strong> mapDamage2 is a computational framework written in Python and R,
which tracks and quantifies DNA damage patterns among ancient DNA sequencing reads
generated by Next-Generation Sequencing platforms.
<strong>Homepage:</strong> https://www.biopython.org</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>mapDamage/2.0.9-GCC-8.2.0-2.31.1-Python-2.7.15</p></li>
<li><p>mapDamage/2.2.0-GCC-8.2.0-2.31.1-Python-3.7.2</p></li>
</ul>
</details>
</details>
<p>###MariaDB-connector-c
<strong>Description:</strong> MariaDB Connector/C is used to connect applications developed in C/C++ to MariaDB and MySQL databases.
<strong>Homepage:</strong> https://downloads.mariadb.org/connector-c/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>MariaDB-connector-c/2.3.7-GCCcore-8.2.0</p></li>
<li><p>MariaDB-connector-c/2.3.7-foss-2018b</p></li>
<li><p>MariaDB-connector-c/2.3.7-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###MATIO
<strong>Description:</strong> matio is an C library for reading and writing Matlab MAT files.
<strong>Homepage:</strong> http://sourceforge.net/projects/matio/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>MATIO/1.5.12-GCCcore-7.3.0</p></li>
</ul>
</details>
</details>
<p>###MATLAB
<strong>Description:</strong> MATLAB is a high-level language and interactive environment
that enables you to perform computationally intensive tasks faster than with
traditional programming languages such as C, C++, and Fortran.
<strong>Homepage:</strong> http://www.mathworks.com/products/matlab</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>MATLAB/2019a</p></li>
</ul>
</details>
</details>
<p>###matplotlib
<strong>Description:</strong> matplotlib is a python 2D plotting library which produces publication quality figures in a variety of
hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python
and ipython shell, web application servers, and six graphical user interface toolkits.
<strong>Homepage:</strong> https://matplotlib.org</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>matplotlib/2.2.3-foss-2018b-Python-2.7.15</p></li>
<li><p>matplotlib/2.2.3-fosscuda-2018b-Python-2.7.15</p></li>
<li><p>matplotlib/2.2.3-intel-2018b-Python-2.7.15</p></li>
<li><p>matplotlib/2.2.4-foss-2019a-Python-2.7.15</p></li>
<li><p>matplotlib/2.2.4-intel-2019a-Python-2.7.15</p></li>
<li><p>matplotlib/3.0.0-foss-2018b-Python-3.6.6</p></li>
<li><p>matplotlib/3.0.0-intel-2018b-Python-3.6.6</p></li>
<li><p>matplotlib/3.0.3-foss-2019a-Python-3.7.2</p></li>
<li><p>matplotlib/3.0.3-intel-2019a-Python-3.7.2</p></li>
<li><p>matplotlib/3.1.1-foss-2019b-Python-3.7.4</p></li>
</ul>
</details>
</details>
<p>###MCL
<strong>Description:</strong> The MCL algorithm is short for the Markov Cluster Algorithm, a fast
and scalable unsupervised cluster algorithm for graphs (also known as networks) based
on simulation of (stochastic) flow in graphs.
<strong>Homepage:</strong> http://micans.org/mcl/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>MCL/14.137-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###MEGAHIT
<strong>Description:</strong> An ultra-fast single-node solution for large and complex
metagenomics assembly via succinct de Bruijn graph
<strong>Homepage:</strong> https://github.com/voutcn/megahit</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>MEGAHIT/1.1.4-foss-2018b-Python-3.6.6</p></li>
<li><p>MEGAHIT/1.2.8-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###Mesa
<strong>Description:</strong> Mesa is an open-source implementation of the OpenGL specification -
a system for rendering interactive 3D graphics.
<strong>Homepage:</strong> https://www.mesa3d.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Mesa/18.1.1-foss-2018b</p></li>
<li><p>Mesa/18.1.1-intel-2018b</p></li>
<li><p>Mesa/19.0.1-GCCcore-8.2.0</p></li>
<li><p>Mesa/19.1.7-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###Meson
<strong>Description:</strong> Meson is a cross-platform build system designed to be both as fast and as user friendly as possible.
<strong>Homepage:</strong> https://mesonbuild.com</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Meson/0.50.0-GCCcore-8.2.0-Python-3.7.2</p></li>
<li><p>Meson/0.51.2-GCCcore-8.3.0-Python-3.7.4</p></li>
</ul>
</details>
</details>
<p>###MetaBAT
<strong>Description:</strong> An efficient tool for accurately reconstructing single genomes from complex microbial communities
<strong>Homepage:</strong> https://bitbucket.org/berkeleylab/metabat</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>MetaBAT/2.14-gompi-2019a</p></li>
</ul>
</details>
</details>
<p>###Metaxa2
<strong>Description:</strong> Metaxa2 – Identifies Small Subunit (SSU) rRNAs and classifies them taxonomically
<strong>Homepage:</strong> https://microbiology.se/software/metaxa2/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Metaxa2/2.2-gompi-2019a</p></li>
</ul>
</details>
</details>
<p>###METIS
<strong>Description:</strong> METIS is a set of serial programs for partitioning graphs, partitioning
finite element meshes, and producing fill reducing orderings for sparse
matrices. The algorithms implemented in METIS are based on the multilevel
recursive-bisection, multilevel k-way, and multi-constraint partitioning
schemes.</p>
<p><strong>Homepage:</strong> http://glaros.dtc.umn.edu/gkhome/metis/metis/overview</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>METIS/5.1.0-GCCcore-7.3.0</p></li>
<li><p>METIS/5.1.0-GCCcore-8.2.0</p></li>
<li><p>METIS/5.1.0-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###Miniconda3
<strong>Description:</strong> Built to complement the rich, open source Python community,
the Anaconda platform provides an enterprise-ready data analytics platform
that empowers companies to adopt a modern open data science analytics architecture.</p>
<p><strong>Homepage:</strong> https://www.continuum.io/anaconda-overview</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Miniconda3/4.5.12</p></li>
<li><p>Miniconda3/4.7.10</p></li>
</ul>
</details>
</details>
<p>###minimap2
<strong>Description:</strong> Minimap2 is a fast sequence mapping and alignment
program that can find overlaps between long noisy reads, or map long
reads or their assemblies to a reference genome optionally with detailed
alignment (i.e. CIGAR). At present, it works efficiently with query
sequences from a few kilobases to ~100 megabases in length at an error
rate ~15%. Minimap2 outputs in the PAF or the SAM format. On limited
test data sets, minimap2 is over 20 times faster than most other
long-read aligners. It will replace BWA-MEM for long reads and contig
alignment.
<strong>Homepage:</strong> https://github.com/lh3/minimap2</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>minimap2/2.13-foss-2018b</p></li>
<li><p>minimap2/2.17-GCC-8.2.0-2.31.1</p></li>
</ul>
</details>
</details>
<p>###MIRA
<strong>Description:</strong> MIRA is a whole genome shotgun and EST sequence assembler for Sanger, 454, Solexa (Illumina),
IonTorrent data and PacBio (the latter at the moment only CCS and error-corrected CLR reads).
<strong>Homepage:</strong> https://sourceforge.net/projects/mira-assembler/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>MIRA/4.9.6-foss-2018b</p></li>
</ul>
</details>
</details>
<p>###MITObim
<strong>Description:</strong> The MITObim procedure (mitochondrial baiting and iterative mapping) represents an efficient approach to assembling novel mitochondrial genomes of non-model organisms directly from total genomic DNA derived NGS reads.
<strong>Homepage:</strong> https://github.com/chrishah/MITObim</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>MITObim/1.9.1-foss-2018b-Perl-5.28.0</p></li>
</ul>
</details>
</details>
<p>###MITOS
<strong>Description:</strong> MITOS is a pipeline designed to compute a consistent de novo annotation of the mitogenomic sequences.
<strong>Homepage:</strong> https://gitlab.com/Bernt/MITOS</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>MITOS/2.0.2-foss-2019a-Python-2.7.15</p></li>
</ul>
</details>
</details>
<p>###Mothur
<strong>Description:</strong> Mothur is a single piece of open-source, expandable software
to fill the bioinformatics needs of the microbial ecology community.
<strong>Homepage:</strong> https://www.mothur.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Mothur/1.41.0-foss-2018b-Python-2.7.15</p></li>
<li><p>Mothur/1.43.0-foss-2019a-Python-3.7.2</p></li>
</ul>
</details>
</details>
<p>###MPFR
<strong>Description:</strong> The MPFR library is a C library for multiple-precision floating-point
computations with correct rounding.</p>
<p><strong>Homepage:</strong> http://www.mpfr.org</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>MPFR/4.0.1-GCCcore-7.3.0</p></li>
<li><p>MPFR/4.0.2-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###MrBayes
<strong>Description:</strong> MrBayes is a program for the Bayesian estimation of phylogeny.
<strong>Homepage:</strong> http://mrbayes.csit.fsu.edu</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>MrBayes/3.2.6-foss-2018b</p></li>
</ul>
</details>
</details>
<p>###MrBayes5D
<strong>Description:</strong> A modified version of MrBayes.
<strong>Homepage:</strong> http://mrbayes.csit.fsu.edu</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>MrBayes5D/3.2.6-foss-2018b</p></li>
</ul>
</details>
</details>
<p>###MSMC
<strong>Description:</strong> This software implements the multiple sequentially Markovian coalescent (MSMC),
a method to infer population size and gene flow from multiple genome sequences.
<strong>Homepage:</strong> https://github.com/stschiff/msmc</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>MSMC/1.1.0-linux64</p></li>
</ul>
</details>
</details>
<p>###MSMC2
<strong>Description:</strong> This program implements MSMC2, a method to infer population
size history and population separation history from whole genome sequencing data.
<strong>Homepage:</strong> https://github.com/stschiff/msmc2</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>MSMC2/2.1.2-linux64</p></li>
</ul>
</details>
</details>
<p>###MUMmer
<strong>Description:</strong> MUMmer is a system for rapidly aligning entire genomes,
whether in complete or draft form. AMOS makes use of it.
<strong>Homepage:</strong> http://mummer.sourceforge.net/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>MUMmer/4.0.0beta2-foss-2018b</p></li>
</ul>
</details>
</details>
<p>###MUSCLE
<strong>Description:</strong> MUSCLE is one of the best-performing multiple alignment programs
according to published benchmark tests, with accuracy and speed that are consistently
better than CLUSTALW. MUSCLE can align hundreds of sequences in seconds. Most users
learn everything they need to know about MUSCLE in a few minutes-only a handful of
command-line options are needed to perform common alignment tasks.
<strong>Homepage:</strong> http://drive5.com/muscle/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>MUSCLE/3.8.31-foss-2018b</p></li>
<li><p>MUSCLE/3.8.31-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###NAMD
<strong>Description:</strong> NAMD is a parallel molecular dynamics code designed for high-performance simulation of
large biomolecular systems.
<strong>Homepage:</strong> http://www.ks.uiuc.edu/Research/namd/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>NAMD/2.13-foss-2018b-mpi</p></li>
</ul>
</details>
</details>
<p>###NASM
<strong>Description:</strong> NASM: General-purpose x86 assembler
<strong>Homepage:</strong> https://www.nasm.us/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>NASM/2.13.03-GCCcore-7.3.0</p></li>
<li><p>NASM/2.14.02-GCCcore-8.2.0</p></li>
<li><p>NASM/2.14.02-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###NBO7
<strong>Description:</strong> NBO7 is the most recent version of a program suite aimed at analysing natural bond order in different outputs: The NBO program performs the analysis of a many-electron molecular wavefunction in terms of localized electron-pair bonding units.
<strong>Homepage:</strong> http://nbo6.chem.wisc.edu</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>NBO7/09-Sep-2019-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###ncbi-vdb
<strong>Description:</strong> The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for
using data in the INSDC Sequence Read Archives.
<strong>Homepage:</strong> https://github.com/ncbi/ncbi-vdb</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ncbi-vdb/2.9.3-foss-2018b</p></li>
</ul>
</details>
</details>
<p>###NCCL
<strong>Description:</strong> The NVIDIA Collective Communications Library (NCCL) implements multi-GPU and multi-node collective
communication primitives that are performance optimized for NVIDIA GPUs.
<strong>Homepage:</strong> https://developer.nvidia.com/nccl</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>NCCL/2.4.8-CUDA-10.0</p></li>
<li><p>NCCL/2.4.8-gcccuda-2018b</p></li>
<li><p>NCCL/2.4.8-gcccuda-2019a</p></li>
<li><p>NCCL/2.4.8-gcccuda-2019b</p></li>
</ul>
</details>
</details>
<p>###ncdf4
<strong>Description:</strong> ncdf4: Interface to Unidata netCDF (version 4 or earlier) format data files
<strong>Homepage:</strong> http://cran.r-project.org/web/packages/ncdf4</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ncdf4/1.16.1-foss-2019a-R-3.6.0</p></li>
</ul>
</details>
</details>
<p>###ncdu
<strong>Description:</strong> Ncdu is a disk usage analyzer with an ncurses interface. It is designed to find space hogs on a
remote server where you don’t have an entire graphical setup available, but it is a useful tool even on regular
desktop systems. Ncdu aims to be fast, simple and easy to use, and should be able to run in any minimal POSIX-like
environment with ncurses installed.
<strong>Homepage:</strong> https://dev.yorhel.nl/ncdu</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ncdu/1.14-GCCcore-7.3.0</p></li>
</ul>
</details>
</details>
<p>###NCO
<strong>Description:</strong> manipulates and analyzes data stored in netCDF-accessible formats, including DAP, HDF4, and HDF5
<strong>Homepage:</strong> http://nco.sourceforge.net</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>NCO/4.7.9-foss-2018b</p></li>
<li><p>NCO/4.7.9-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###ncurses
<strong>Description:</strong> The Ncurses (new curses) library is a free software emulation of curses in System V Release 4.0,
and more. It uses Terminfo format, supports pads and color and multiple highlights and forms characters and
function-key mapping, and has all the other SYSV-curses enhancements over BSD Curses.
<strong>Homepage:</strong> http://www.gnu.org/software/ncurses/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ncurses/6.0</p></li>
<li><p>ncurses/6.0-GCCcore-7.3.0</p></li>
<li><p>ncurses/6.1</p></li>
<li><p>ncurses/6.1-GCCcore-7.3.0</p></li>
<li><p>ncurses/6.1-GCCcore-8.2.0</p></li>
<li><p>ncurses/6.1-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###ncview
<strong>Description:</strong> Ncview is a visual browser for netCDF format files.
Typically you would use ncview to get a quick and easy, push-button
look at your netCDF files. You can view simple movies of the data,
view along various dimensions, take a look at the actual data values,
change color maps, invert the data, etc.
<strong>Homepage:</strong> http://meteora.ucsd.edu/~pierce/ncview_home_page.html</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ncview/2.1.7-foss-2018b</p></li>
<li><p>ncview/2.1.7-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###netCDF
<strong>Description:</strong> NetCDF (network Common Data Form) is a set of software libraries
and machine-independent data formats that support the creation, access, and sharing of array-oriented
scientific data.
<strong>Homepage:</strong> https://www.unidata.ucar.edu/software/netcdf/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>netCDF/4.6.1-foss-2018b</p></li>
<li><p>netCDF/4.6.1-fosscuda-2018b</p></li>
<li><p>netCDF/4.6.1-intel-2018b</p></li>
<li><p>netCDF/4.6.2-gompi-2019a</p></li>
<li><p>netCDF/4.6.2-iimpi-2019a</p></li>
<li><p>netCDF/4.7.1-gompi-2019b</p></li>
<li><p>netCDF/4.7.1-iimpi-2019b</p></li>
</ul>
</details>
</details>
<p>###netCDF-Fortran
<strong>Description:</strong> NetCDF (network Common Data Form) is a set of software libraries
and machine-independent data formats that support the creation, access, and sharing of array-oriented
scientific data.
<strong>Homepage:</strong> http://www.unidata.ucar.edu/software/netcdf/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>netCDF-Fortran/4.4.4-foss-2018b</p></li>
<li><p>netCDF-Fortran/4.4.4-fosscuda-2018b</p></li>
<li><p>netCDF-Fortran/4.4.4-intel-2018b</p></li>
<li><p>netCDF-Fortran/4.4.5-gompi-2019a</p></li>
<li><p>netCDF-Fortran/4.4.5-iimpi-2019a</p></li>
</ul>
</details>
</details>
<p>###netcdf4-python
<strong>Description:</strong> Python/numpy interface to netCDF.
<strong>Homepage:</strong> https://unidata.github.io/netcdf4-python/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>netcdf4-python/1.4.1-foss-2018b-Python-3.6.6</p></li>
<li><p>netcdf4-python/1.4.1-intel-2018b-Python-3.6.6</p></li>
</ul>
</details>
</details>
<p>###nettle
<strong>Description:</strong> Nettle is a cryptographic library that is designed to fit easily
in more or less any context: In crypto toolkits for object-oriented
languages (C++, Python, Pike, …), in applications like LSH or GNUPG,
or even in kernel space.
<strong>Homepage:</strong> http://www.lysator.liu.se/~nisse/nettle/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>nettle/3.4-foss-2018b</p></li>
<li><p>nettle/3.4-intel-2018b</p></li>
<li><p>nettle/3.4.1-GCCcore-8.2.0</p></li>
<li><p>nettle/3.5.1-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###NGS
<strong>Description:</strong> NGS is a new, domain-specific API for accessing reads, alignments and pileups produced from
Next Generation Sequencing.
<strong>Homepage:</strong> https://github.com/ncbi/ngs</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>NGS/2.9.3-foss-2018b-Java-1.8</p></li>
</ul>
</details>
</details>
<p>###NGSadmix
<strong>Description:</strong> NGSadmix is a tool for finding admixture proportions
from NGS data, based on genotype likelihoods.
<strong>Homepage:</strong> http://www.popgen.dk/software/index.php/NgsAdmix</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>NGSadmix/32-GCC-7.3.0-2.30</p></li>
</ul>
</details>
</details>
<p>###ngsDist
<strong>Description:</strong> A program to estimate pairwise genetic distances directly,
taking the uncertainty of genotype’s assignation into account.
<strong>Homepage:</strong> https://github.com/fgvieira/ngsDist</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ngsDist/191118-GCC-8.2.0-2.31.1</p></li>
</ul>
</details>
</details>
<p>###ngsLD
<strong>Description:</strong> a program to estimate pairwise linkage disequilibrium (LD)
taking the uncertainty of genotype’s assignation into account.
<strong>Homepage:</strong> https://github.com/fgvieira/ngsLD</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ngsLD/191108-GCC-8.2.0-2.31.1</p></li>
</ul>
</details>
</details>
<p>###ngsTools
<strong>Description:</strong> A collection of programs for population genetics analyses from NGS data,
taking into account data statistical uncertainty.
<strong>Homepage:</strong> https://github.com/mfumagalli/ngsTools</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ngsTools/190624-GCC-8.2.0-2.31.1</p></li>
</ul>
</details>
</details>
<p>###Ninja
<strong>Description:</strong> Ninja is a small build system with a focus on speed.
<strong>Homepage:</strong> https://ninja-build.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Ninja/1.8.2-fosscuda-2018b</p></li>
<li><p>Ninja/1.9.0-GCCcore-8.2.0</p></li>
<li><p>Ninja/1.9.0-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###NLopt
<strong>Description:</strong> NLopt is a free/open-source library for nonlinear optimization,
providing a common interface for a number of different free optimization routines
available online as well as original implementations of various other algorithms.
<strong>Homepage:</strong> http://ab-initio.mit.edu/wiki/index.php/NLopt</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>NLopt/2.4.2-GCCcore-7.3.0</p></li>
<li><p>NLopt/2.6.1-GCCcore-8.2.0</p></li>
<li><p>NLopt/2.6.1-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###NOVOPlasty
<strong>Description:</strong> NOVOPlasty is a de novo assembler and heteroplasmy/variance caller for short circular genomes.
<strong>Homepage:</strong> https://github.com/ndierckx/NOVOPlasty</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>NOVOPlasty/3.7-GCCcore-8.3.0</p></li>
<li><p>NOVOPlasty/3.7-intel-2018b-Perl-5.28.0</p></li>
</ul>
</details>
</details>
<p>###NSPR
<strong>Description:</strong> Netscape Portable Runtime (NSPR) provides a platform-neutral API for system level
and libc-like functions.
<strong>Homepage:</strong> https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSPR</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>NSPR/4.20-GCCcore-7.3.0</p></li>
<li><p>NSPR/4.21-GCCcore-8.2.0</p></li>
<li><p>NSPR/4.21-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###NSS
<strong>Description:</strong> Network Security Services (NSS) is a set of libraries designed to support cross-platform development
of security-enabled client and server applications.
<strong>Homepage:</strong> https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>NSS/3.39-GCCcore-7.3.0</p></li>
<li><p>NSS/3.42.1-GCCcore-8.2.0</p></li>
<li><p>NSS/3.45-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###ntCard
<strong>Description:</strong> ntCard is a streaming algorithm for cardinality estimation in genomics datasets.
<strong>Homepage:</strong> https://github.com/bcgsc/ntCard</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ntCard/1.1.1-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###ntEdit
<strong>Description:</strong> Fast, lightweight, scalable genome sequence polishing &amp; snv detection.
<strong>Homepage:</strong> https://github.com/bcgsc/ntEdit</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ntEdit/1.3.1-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###ntHits
<strong>Description:</strong> ntHits is a method for identifying reapeats in high-throughput DNA sequencing data.
<strong>Homepage:</strong> https://github.com/bcgsc/ntHits</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ntHits/0.0.1-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###numactl
<strong>Description:</strong> The numactl program allows you to run your application program on specific
cpu’s and memory nodes. It does this by supplying a NUMA memory policy to
the operating system before running your program. The libnuma library provides
convenient ways for you to add NUMA memory policies into your own program.</p>
<p><strong>Homepage:</strong> http://oss.sgi.com/projects/libnuma/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>numactl/2.0.11-GCCcore-7.3.0</p></li>
<li><p>numactl/2.0.12-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###numba
<strong>Description:</strong> Numba is an Open Source NumPy-aware optimizing compiler for Python sponsored by Continuum Analytics,
Inc. It uses the remarkable LLVM compiler infrastructure to compile Python syntax to machine code.
<strong>Homepage:</strong> http://numba.pydata.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>numba/0.41.0-foss-2018b-Python-3.6.6</p></li>
<li><p>numba/0.41.0-intel-2018b-Python-2.7.15</p></li>
</ul>
</details>
</details>
<p>###numexpr
<strong>Description:</strong> The numexpr package evaluates multiple-operator array expressions many times faster than NumPy can.
It accepts the expression as a string, analyzes it, rewrites it more efficiently, and compiles it on the fly into
code for its internal virtual machine (VM). Due to its integrated just-in-time (JIT) compiler, it does not require a
compiler at runtime.
<strong>Homepage:</strong> https://numexpr.readthedocs.io</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>numexpr/2.7.0-foss-2019a</p></li>
</ul>
</details>
</details>
<p>###NVidia
<strong>Description:</strong> This module makes NVidia drivers available to EasyBuild softwawre.
<strong>Homepage:</strong> https://www.nvidia.com/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>NVidia/1.0</p></li>
</ul>
</details>
</details>
<p>###NWChem
<strong>Description:</strong> NWChem aims to provide its users with computational chemistry tools that are scalable both in
their ability to treat large scientific computational chemistry problems efficiently, and in their use of available
parallel computing resources from high-performance parallel supercomputers to conventional workstation clusters.
NWChem software can handle: biomolecules, nanostructures, and solid-state; from quantum to classical, and all
combinations; Gaussian basis functions or plane-waves; scaling from one to thousands of processors; properties
and relativity.
<strong>Homepage:</strong> http://www.nwchem-sw.org</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>NWChem/6.8.1.revision133-intel-2018b-2018-06-14-Python-2.7.15</p></li>
</ul>
</details>
</details>
<p>###OBITools
<strong>Description:</strong> The OBITools package is a set of programs specifically designed for analyzing NGS
data in a DNA metabarcoding context, taking into account taxonomic information.
<strong>Homepage:</strong> https://pythonhosted.org/OBITools/welcome.html</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>OBITools/1.2.12-foss-2018b-Python-2.7.15</p></li>
</ul>
</details>
</details>
<p>###OpenBLAS
<strong>Description:</strong> OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version.
<strong>Homepage:</strong> http://xianyi.github.com/OpenBLAS/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>OpenBLAS/0.3.1-GCC-7.3.0-2.30</p></li>
<li><p>OpenBLAS/0.3.5-GCC-8.2.0-2.31.1</p></li>
<li><p>OpenBLAS/0.3.7-GCC-8.3.0</p></li>
</ul>
</details>
</details>
<p>###OpenFOAM
<strong>Description:</strong> OpenFOAM is a free, open source CFD software package.
OpenFOAM has an extensive range of features to solve anything from complex fluid flows
involving chemical reactions, turbulence and heat transfer,
to solid dynamics and electromagnetics.
<strong>Homepage:</strong> http://www.openfoam.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>OpenFOAM/6-foss-2018b</p></li>
</ul>
</details>
</details>
<p>###OpenMPI
<strong>Description:</strong> The Open MPI Project is an open source MPI-3 implementation.
<strong>Homepage:</strong> https://www.open-mpi.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>OpenMPI/3.1.1-GCC-7.3.0-2.30</p></li>
<li><p>OpenMPI/3.1.1-gcccuda-2018b</p></li>
<li><p>OpenMPI/3.1.1-iccifort-2018.3.222-GCC-7.3.0-2.30</p></li>
<li><p>OpenMPI/3.1.3-GCC-8.2.0-2.31.1</p></li>
<li><p>OpenMPI/3.1.3-GCC-8.2.0-2.31.1-hpcx_2.4</p></li>
<li><p>OpenMPI/3.1.3-GCC-8.2.0-2.31.1-hpcx_2.5</p></li>
<li><p>OpenMPI/3.1.4-GCC-8.3.0</p></li>
<li><p>OpenMPI/3.1.4-PGI-19.9-GCC-8.3.0</p></li>
</ul>
</details>
</details>
<p>###OpenPGM
<strong>Description:</strong> OpenPGM is an open source implementation of the Pragmatic General Multicast
(PGM) specification in RFC 3208 available at www.ietf.org. PGM is a reliable
and scalable multicast protocol that enables receivers to detect loss, request
retransmission of lost data, or notify an application of unrecoverable loss.
PGM is a receiver-reliable protocol, which means the receiver is responsible
for ensuring all data is received, absolving the sender of reception
responsibility.</p>
<p><strong>Homepage:</strong> http://code.google.com/p/openpgm/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>OpenPGM/5.2.122-GCCcore-7.3.0</p></li>
</ul>
</details>
</details>
<p>###ORCA
<strong>Description:</strong> ORCA is a flexible, efficient and easy-to-use general purpose tool for quantum chemistry
with specific emphasis on spectroscopic properties of open-shell molecules.
It features a wide variety of standard quantum chemical methods ranging from semiempirical methods to DFT to single-
and multireference correlated ab initio methods.
It can also treat environmental and relativistic effects.
<strong>Homepage:</strong> https://orcaforum.kofo.mpg.de</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ORCA/4.2.0-gompi-2019b</p></li>
</ul>
</details>
</details>
<p>###OrthoMCL
<strong>Description:</strong> OrthoMCL is a genome-scale algorithm for grouping orthologous protein sequences.
<strong>Homepage:</strong> http://orthomcl.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>OrthoMCL/2.0.9-intel-2018b-Perl-5.28.0</p></li>
</ul>
</details>
</details>
<p>###p7zip
<strong>Description:</strong> p7zip is a quick port of 7z.exe and 7za.exe (command line version of 7zip) for Unix.
7-Zip is a file archiver with highest compression ratio.
<strong>Homepage:</strong> http://p7zip.sourceforge.net/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>p7zip/16.02-intel-2018a</p></li>
</ul>
</details>
</details>
<p>###PALEOMIX
<strong>Description:</strong> The PALEOMIX pipelines are a set of pipelines and tools designed to aid the rapid processing of High-Throughput Sequencing (HTS) data.
<strong>Homepage:</strong> https://github.com/MikkelSchubert/paleomix</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>PALEOMIX/1.2.14-GCC-8.2.0-2.31.1-Python-2.7.15</p></li>
</ul>
</details>
</details>
<p>###PAML
<strong>Description:</strong> PAML is a package of programs for phylogenetic
analyses of DNA or protein sequences using maximum likelihood.
<strong>Homepage:</strong> http://abacus.gene.ucl.ac.uk/software/paml.html</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>PAML/4.9i-GCC-7.3.0-2.30</p></li>
</ul>
</details>
</details>
<p>###Pango
<strong>Description:</strong> Pango is a library for laying out and rendering of text, with an emphasis on internationalization.
Pango can be used anywhere that text layout is needed, though most of the work on Pango so far has been done in the
context of the GTK+ widget toolkit. Pango forms the core of text and font handling for GTK+-2.x.
<strong>Homepage:</strong> http://www.pango.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Pango/1.42.4-foss-2018b</p></li>
<li><p>Pango/1.43.0-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###parallel
<strong>Description:</strong> parallel: Build and execute shell commands in parallel
<strong>Homepage:</strong> https://savannah.gnu.org/projects/parallel/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>parallel/20190622-GCCcore-8.2.0</p></li>
<li><p>parallel/20190922-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###ParaView
<strong>Description:</strong> ParaView is a scientific parallel visualizer.
<strong>Homepage:</strong> http://www.paraview.org</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ParaView/5.4.1-foss-2018b-mpi</p></li>
<li><p>ParaView/5.5.2-foss-2018b-mpi</p></li>
</ul>
</details>
</details>
<p>###ParMETIS
<strong>Description:</strong> ParMETIS is an MPI-based parallel library that implements a variety of algorithms for partitioning
unstructured graphs, meshes, and for computing fill-reducing orderings of sparse matrices. ParMETIS extends the
functionality provided by METIS and includes routines that are especially suited for parallel AMR computations and
large scale numerical simulations. The algorithms implemented in ParMETIS are based on the parallel multilevel k-way
graph-partitioning, adaptive repartitioning, and parallel multi-constrained partitioning schemes.
<strong>Homepage:</strong> http://glaros.dtc.umn.edu/gkhome/metis/parmetis/overview</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ParMETIS/4.0.3-foss-2018b</p></li>
<li><p>ParMETIS/4.0.3-gompi-2019a</p></li>
<li><p>ParMETIS/4.0.3-iimpi-2019a</p></li>
<li><p>ParMETIS/4.0.3-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###parsyncfp
<strong>Description:</strong> Parallel rsync wrapper for large data transfers
<strong>Homepage:</strong> http://moo.nac.uci.edu/~hjm/parsync/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>parsyncfp/1.61</p></li>
</ul>
</details>
</details>
<p>###patchelf
<strong>Description:</strong> PatchELF is a small utility to modify the dynamic linker and RPATH of ELF executables.
<strong>Homepage:</strong> http://nixos.org/patchelf.html</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>patchelf/0.9-GCCcore-7.3.0</p></li>
<li><p>patchelf/0.10-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###PCAngsd
<strong>Description:</strong> PCAngsd, which estimates the covariance matrix for low depth
NGS data in an iterative procedure based on genotype likelihoods and is able
to perform multiple population genetic analyses in heterogeneous populations.
<strong>Homepage:</strong> http://www.popgen.dk/software/index.php/PCAngsd</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>PCAngsd/0.98-foss-2019a-Python-2.7.15</p></li>
<li><p>PCAngsd/200115-foss-2019a-Python-2.7.15</p></li>
</ul>
</details>
</details>
<p>###PCRE
<strong>Description:</strong> The PCRE library is a set of functions that implement regular expression
pattern matching using the same syntax and semantics as Perl 5.</p>
<p><strong>Homepage:</strong> https://www.pcre.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>PCRE/8.41-GCCcore-7.3.0</p></li>
<li><p>PCRE/8.43-GCCcore-8.2.0</p></li>
<li><p>PCRE/8.43-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###PCRE2
<strong>Description:</strong> The PCRE library is a set of functions that implement regular expression pattern matching using the same syntax
and semantics as Perl 5.</p>
<p><strong>Homepage:</strong> https://www.pcre.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>PCRE2/10.33-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###Perl
<strong>Description:</strong> Larry Wall’s Practical Extraction and Report Language
<strong>Homepage:</strong> https://www.perl.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Perl/5.28.0-GCCcore-7.3.0</p></li>
<li><p>Perl/5.28.1-GCCcore-8.2.0</p></li>
<li><p>Perl/5.30.0-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###PETSc
<strong>Description:</strong> PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the
scalable (parallel) solution of scientific applications modeled by partial differential equations.
<strong>Homepage:</strong> http://www.mcs.anl.gov/petsc</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>PETSc/3.11.0-foss-2018b</p></li>
<li><p>PETSc/3.11.0-intel-2018b</p></li>
<li><p>PETSc/3.11.1-foss-2018b</p></li>
<li><p>PETSc/3.11.1-foss-2019a</p></li>
<li><p>PETSc/3.11.1-intel-2018b</p></li>
<li><p>PETSc/3.11.1-intel-2019a</p></li>
</ul>
</details>
</details>
<p>###petsc4py
<strong>Description:</strong> petsc4py are Python bindings for PETSc, the Portable, Extensible Toolchain for Scientific Computation.
<strong>Homepage:</strong> https://bitbucket.org/petsc/petsc4py</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>petsc4py/3.11.0-foss-2018b-Python-3.6.6</p></li>
</ul>
</details>
</details>
<p>###PGDSpider
<strong>Description:</strong> An automated data conversion tool for connecting population genetics and genomics programs
<strong>Homepage:</strong> http://cmpg.unibe.ch/software/PGDSpider/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>PGDSpider/2.1.1.5-Java-1.8</p></li>
</ul>
</details>
</details>
<p>###PGI
<strong>Description:</strong> C, C++ and Fortran compilers from The Portland Group
<strong>Homepage:</strong> http://www.pgroup.com/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>PGI/19.9-GCC-8.3.0</p></li>
</ul>
</details>
</details>
<p>###phonopy
<strong>Description:</strong> Phonopy is an open source package of phonon calculations based on the supercell approach.
<strong>Homepage:</strong> http://atztogo.github.io/phonopy/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>phonopy/1.14.2-intel-2018b-Python-3.6.6</p></li>
<li><p>phonopy/2.2.0-intel-2019a-Python-3.7.2</p></li>
</ul>
</details>
</details>
<p>###PhyloBayes-MPI
<strong>Description:</strong> A Bayesian software for phylogenetic reconstruction using mixture models
<strong>Homepage:</strong> https://github.com/bayesiancook/pbmpi</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>PhyloBayes-MPI/1.8-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###picard
<strong>Description:</strong> A set of tools (in Java) for working with next generation sequencing data in the BAM format.
<strong>Homepage:</strong> https://sourceforge.net/projects/picard</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>picard/2.18.27-Java-1.8</p></li>
<li><p>picard/2.21.1-Java-11</p></li>
</ul>
</details>
</details>
<p>###pigz
<strong>Description:</strong> pigz, which stands for parallel implementation of gzip, is a fully
functional replacement for gzip that exploits multiple processors and multiple
cores to the hilt when compressing data. pigz was written by Mark Adler, and
uses the zlib and pthread libraries.</p>
<p><strong>Homepage:</strong> http://zlib.net/pigz/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>pigz/2.4-GCCcore-7.3.0</p></li>
<li><p>pigz/2.4-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###Pillow
<strong>Description:</strong> Pillow is the ‘friendly PIL fork’ by Alex Clark and Contributors.
PIL is the Python Imaging Library by Fredrik Lundh and Contributors.
<strong>Homepage:</strong> http://pillow.readthedocs.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Pillow/5.3.0-foss-2018b-Python-3.6.6</p></li>
<li><p>Pillow/5.3.0-intel-2018b-Python-2.7.15</p></li>
<li><p>Pillow/6.0.0-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###Pilon
<strong>Description:</strong> Pilon is an automated genome assembly improvement and variant detection tool
<strong>Homepage:</strong> https://github.com/broadinstitute/pilon</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Pilon/1.23-Java-11</p></li>
</ul>
</details>
</details>
<p>###Pindel
<strong>Description:</strong> Pindel can detect breakpoints of large deletions, medium sized
insertions, inversions, tandem duplications and other structural variants at single-based
resolution from next-gen sequence data. It uses a pattern growth approach to identify the
breakpoints of these variants from paired-end short reads.
<strong>Homepage:</strong> http://gmt.genome.wustl.edu/packages/pindel/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Pindel/0.2.5b9-20170508-GCC-7.3.0-2.30</p></li>
</ul>
</details>
</details>
<p>###pixman
<strong>Description:</strong> Pixman is a low-level software library for pixel manipulation, providing
features such as image compositing and trapezoid rasterization. Important
users of pixman are the cairo graphics library and the X server.</p>
<p><strong>Homepage:</strong> http://www.pixman.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>pixman/0.34.0-GCCcore-7.3.0</p></li>
<li><p>pixman/0.38.0-GCCcore-8.2.0</p></li>
<li><p>pixman/0.38.4-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###pkg-config
<strong>Description:</strong> pkg-config is a helper tool used when compiling applications and libraries.
It helps you insert the correct compiler options on the command line so an
application can use gcc -o test test.c <code class="docutils literal notranslate"><span class="pre">pkg-config</span> <span class="pre">--libs</span> <span class="pre">--cflags</span> <span class="pre">glib-2.0</span></code>
for instance, rather than hard-coding values on where to find glib (or other
libraries).</p>
<p><strong>Homepage:</strong> https://www.freedesktop.org/wiki/Software/pkg-config/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>pkg-config/0.29.2-GCCcore-7.3.0</p></li>
<li><p>pkg-config/0.29.2-GCCcore-8.2.0</p></li>
<li><p>pkg-config/0.29.2-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###pkgconfig
<strong>Description:</strong> pkgconfig is a Python module to interface with the pkg-config command line tool
<strong>Homepage:</strong> https://github.com/matze/pkgconfig</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>pkgconfig/1.3.1-foss-2018b-Python-3.6.6</p></li>
<li><p>pkgconfig/1.3.1-fosscuda-2018b-Python-3.6.6</p></li>
<li><p>pkgconfig/1.3.1-intel-2018b-Python-3.6.6</p></li>
<li><p>pkgconfig/1.5.1-GCCcore-8.2.0-python</p></li>
<li><p>pkgconfig/1.5.1-GCCcore-8.3.0-Python-3.7.4</p></li>
</ul>
</details>
</details>
<p>###PLINK
<strong>Description:</strong> Whole-genome association analysis toolset
<strong>Homepage:</strong> https://www.cog-genomics.org/plink/1.9/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>PLINK/1.9b_6.13-x86_64</p></li>
<li><p>PLINK/2.00-alpha1-x86_64</p></li>
</ul>
</details>
</details>
<p>###PLUMED
<strong>Description:</strong> PLUMED is an open source library for free energy calculations in molecular systems which
works together with some of the most popular molecular dynamics engines. Free energy calculations can be
performed as a function of many order parameters with a particular  focus on biological problems, using
state of the art methods such as metadynamics, umbrella sampling and Jarzynski-equation based steered MD.
The software, written in C++, can be easily interfaced with both fortran and C/C++ codes.</p>
<p><strong>Homepage:</strong> http://www.plumed-code.org</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>PLUMED/2.4.0-intel-2018a</p></li>
<li><p>PLUMED/2.5.0-foss-2018b-Python-2.7.15</p></li>
<li><p>PLUMED/2.5.0-fosscuda-2018b-Python-2.7.15</p></li>
<li><p>PLUMED/2.5.1-foss-2019a</p></li>
<li><p>PLUMED/2.5.1-intel-2019a</p></li>
</ul>
</details>
</details>
<p>###PLY
<strong>Description:</strong> PLY is yet another implementation of lex and yacc for Python.
<strong>Homepage:</strong> http://www.dabeaz.com/ply/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>PLY/3.11-foss-2018b-Python-3.6.6</p></li>
</ul>
</details>
</details>
<p>###pompi
<strong>Description:</strong> Toolchain with PGI C, C++ and Fortran compilers, alongside OpenMPI.
<strong>Homepage:</strong> http://www.pgroup.com/index.htm</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>pompi/2019.09</p></li>
</ul>
</details>
</details>
<p>###PoPoolation2
<strong>Description:</strong> PoPoolation2 allows to compare allele frequencies for SNPs between two or more populations and to identify significant differences.
<strong>Homepage:</strong> https://sourceforge.net/p/popoolation2/wiki/Main/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>PoPoolation2/1201-foss-2019a</p></li>
</ul>
</details>
</details>
<p>###pplacer
<strong>Description:</strong> Pplacer places query sequences on a fixed reference phylogenetic tree to maximize phylogenetic
likelihood or posterior probability according to a reference alignment. Pplacer is designed to be fast, to give
useful information about uncertainty, and to offer advanced visualization and downstream analysis.
<strong>Homepage:</strong> http://matsen.fhcrc.org/pplacer/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>pplacer/1.1.alpha19</p></li>
</ul>
</details>
</details>
<p>###preseq
<strong>Description:</strong> Software for predicting library complexity and genome coverage in high-throughput sequencing.
<strong>Homepage:</strong> http://smithlabresearch.org/software/preseq</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>preseq/2.0.3-foss-2018b</p></li>
</ul>
</details>
</details>
<p>###prodigal
<strong>Description:</strong> Prodigal (Prokaryotic Dynamic Programming Genefinding Algorithm)
is a microbial (bacterial and archaeal) gene finding program developed
at Oak Ridge National Laboratory and the University of Tennessee.
<strong>Homepage:</strong> https://github.com/hyattpd/Prodigal/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>prodigal/2.6.3-GCCcore-7.3.0</p></li>
<li><p>prodigal/2.6.3-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###PROJ
<strong>Description:</strong> Program proj is a standard Unix filter function which converts
geographic longitude and latitude coordinates into cartesian coordinates
<strong>Homepage:</strong> https://proj.org</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>PROJ/5.0.0-foss-2018b</p></li>
<li><p>PROJ/5.0.0-intel-2018b</p></li>
<li><p>PROJ/6.0.0-GCCcore-8.2.0</p></li>
<li><p>PROJ/6.2.1-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###prokka
<strong>Description:</strong> Prokka is a software tool for the rapid annotation of prokaryotic genomes.
<strong>Homepage:</strong> https://www.vicbioinformatics.com/software.prokka.shtml</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>prokka/1.13.4-foss-2018b</p></li>
<li><p>prokka/1.14.5-gompi-2019b</p></li>
</ul>
</details>
</details>
<p>###protobuf
<strong>Description:</strong> Google Protocol Buffers
<strong>Homepage:</strong> https://github.com/google/protobuf/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>protobuf/3.6.1-GCCcore-7.3.0</p></li>
<li><p>protobuf/3.10.0-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###psmc
<strong>Description:</strong> This software package infers population size history from a diploid sequence
using the Pairwise Sequentially Markovian Coalescent (PSMC) model.
<strong>Homepage:</strong> https://github.com/lh3/psmc</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>psmc/0.6.5-foss-2018b</p></li>
</ul>
</details>
</details>
<p>###Purge_Haplotigs
<strong>Description:</strong> Pipeline to help with curating heterozygous diploid genome assemblies
<strong>Homepage:</strong> https://bitbucket.org/mroachawri/purge_haplotigs/src/master/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Purge_Haplotigs/1.0.4-foss-2018b-Perl-5.28.0</p></li>
</ul>
</details>
</details>
<p>###pybind11
<strong>Description:</strong> pybind11 is a lightweight header-only library that exposes C++ types in Python and vice versa,
mainly to create Python bindings of existing C++ code.
<strong>Homepage:</strong> https://pybind11.readthedocs.io</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>pybind11/2.2.4-foss-2018b-Python-3.6.6</p></li>
<li><p>pybind11/2.4.3-GCCcore-8.3.0-Python-3.7.4</p></li>
</ul>
</details>
</details>
<p>###Pysam
<strong>Description:</strong> Pysam is a python module for reading and manipulating Samfiles.
It’s a lightweight wrapper of the samtools C-API. Pysam also includes an interface for tabix.
<strong>Homepage:</strong> https://github.com/pysam-developers/pysam</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Pysam/0.15.2-GCC-8.2.0-2.31.1</p></li>
<li><p>Pysam/0.15.2-iccifort-2019.1.144-GCC-8.2.0-2.31.1</p></li>
<li><p>Pysam/0.15.3-GCC-8.3.0</p></li>
<li><p>Pysam/0.15.3-iccifort-2019.5.281</p></li>
</ul>
</details>
</details>
<p>###PyTables
<strong>Description:</strong> PyTables is a package for managing hierarchical datasets and designed to efficiently and easily cope
with extremely large amounts of data. PyTables is built on top of the HDF5 library, using the Python language and the
NumPy package. It features an object-oriented interface that, combined with C extensions for the performance-critical
parts of the code (generated using Cython), makes it a fast, yet extremely easy to use tool for interactively browse,
process and search very large amounts of data. One important feature of PyTables is that it optimizes memory and disk
resources so that data takes much less space (specially if on-flight compression is used) than other solutions such as
relational or object oriented databases.
<strong>Homepage:</strong> http://www.pytables.org</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>PyTables/3.5.2-foss-2019a</p></li>
</ul>
</details>
</details>
<p>###pytest
<strong>Description:</strong> pytest: simple powerful testing with Python
<strong>Homepage:</strong> http://pytest.org</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>pytest/4.4.0-foss-2018b-Python-3.6.6</p></li>
</ul>
</details>
</details>
<p>###Python
<strong>Description:</strong> Python is a programming language that lets you work more quickly and integrate your systems
more effectively.
<strong>Homepage:</strong> http://python.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Python/2.7.15-GCCcore-7.3.0-bare</p></li>
<li><p>Python/2.7.15-GCCcore-8.2.0</p></li>
<li><p>Python/2.7.15-foss-2018b</p></li>
<li><p>Python/2.7.15-fosscuda-2018b</p></li>
<li><p>Python/2.7.15-intel-2018b</p></li>
<li><p>Python/2.7.16-GCCcore-8.3.0</p></li>
<li><p>Python/3.6.6-foss-2018b</p></li>
<li><p>Python/3.6.6-fosscuda-2018b</p></li>
<li><p>Python/3.6.6-intel-2018b</p></li>
<li><p>Python/3.7.2-GCCcore-8.2.0</p></li>
<li><p>Python/3.7.4-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###PyTorch
<strong>Description:</strong> Tensors and Dynamic neural networks in Python with strong GPU acceleration.
PyTorch is a deep learning framework that puts Python first.
<strong>Homepage:</strong> http://pytorch.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>PyTorch/1.0.1-fosscuda-2018b-Python-3.6.6</p></li>
</ul>
</details>
</details>
<p>###PyYAML
<strong>Description:</strong> PyYAML is a YAML parser and emitter for the Python programming language.
<strong>Homepage:</strong> https://pypi.python.org/pypi/PyYAML/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>PyYAML/3.13-fosscuda-2018b-Python-3.6.6</p></li>
<li><p>PyYAML/3.13-intel-2018b-Python-3.6.6</p></li>
<li><p>PyYAML/5.1-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###QIIME2
<strong>Description:</strong> QIIME is an open-source bioinformatics pipeline for performing microbiome analysis
from raw DNA sequencing data.
<strong>Homepage:</strong> http://qiime2.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>QIIME2/2019.4</p></li>
<li><p>QIIME2/2019.7</p></li>
</ul>
</details>
</details>
<p>###Qt5
<strong>Description:</strong> Qt is a comprehensive cross-platform C++ application framework.
<strong>Homepage:</strong> https://qt.io/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Qt5/5.10.1-foss-2018b</p></li>
<li><p>Qt5/5.11.2-foss-2018b-nvidia</p></li>
<li><p>Qt5/5.12.3-GCCcore-8.2.0</p></li>
<li><p>Qt5/5.13.1-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###QuantumESPRESSO
<strong>Description:</strong> Quantum ESPRESSO  is an integrated suite of computer codes
for electronic-structure calculations and materials modeling at the nanoscale.
It is based on density-functional theory, plane waves, and pseudopotentials
(both norm-conserving and ultrasoft).</p>
<p><strong>Homepage:</strong> https://www.quantum-espresso.org</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>QuantumESPRESSO/6.4.1-intel-2019a</p></li>
</ul>
</details>
</details>
<p>###QUAST
<strong>Description:</strong> QUAST evaluates genome assemblies by computing various metrics.
It works both with and without reference genomes. The tool accepts multiple
assemblies, thus is suitable for comparison.
<strong>Homepage:</strong> https://github.com/ablab/quast</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>QUAST/5.0.2-foss-2018b-Python-2.7.15</p></li>
</ul>
</details>
</details>
<p>###Qwt
<strong>Description:</strong> The Qwt library contains GUI Components and utility classes which are primarily useful for programs
with a technical background.
<strong>Homepage:</strong> http://qwt.sourceforge.net/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Qwt/6.1.4-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###R
<strong>Description:</strong> R is a free software environment for statistical computing
and graphics.
<strong>Homepage:</strong> https://www.r-project.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>R/3.5.1-foss-2018b</p></li>
<li><p>R/3.5.1-intel-2018b</p></li>
<li><p>R/3.6.0-foss-2019a</p></li>
<li><p>R/3.6.0-intel-2019a</p></li>
<li><p>R/3.6.2-foss-2019b</p></li>
</ul>
</details>
</details>
<p>###R-bundle-Bioconductor
<strong>Description:</strong> R is a free software environment for statistical computing and graphics.
<strong>Homepage:</strong> http://www.r-project.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>R-bundle-Bioconductor/3.8-foss-2018b-R-3.5.1</p></li>
<li><p>R-bundle-Bioconductor/3.8-intel-2018b-R-3.5.1</p></li>
<li><p>R-bundle-Bioconductor/3.9-foss-2019a-R-3.6.0</p></li>
</ul>
</details>
</details>
<p>###RADProc
<strong>Description:</strong> An algorithm and software package that streamlines and
accelerates de novo locus formation and catalog building by eliminating
redundancies and using a highly efficient method for nucleotide distance
calculation.
<strong>Homepage:</strong> https://github.com/beiko-lab/RADProc</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>RADProc/191206-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###RAxML
<strong>Description:</strong> RAxML search algorithm for maximum likelihood based inference of phylogenetic trees.
<strong>Homepage:</strong> https://github.com/stamatak/standard-RAxML</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>RAxML/8.2.12-foss-2018b-hybrid-avx2</p></li>
<li><p>RAxML/8.2.12-intel-2018b-hybrid-avx2</p></li>
</ul>
</details>
</details>
<p>###RAxML-NG
<strong>Description:</strong> A phylogenetic tree inference tool which uses
maximum-likelihood (ML) optimality criterion.
<strong>Homepage:</strong> https://github.com/amkozlov/raxml-ng</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>RAxML-NG/191127-foss-2019a</p></li>
</ul>
</details>
</details>
<p>###re2c
<strong>Description:</strong> re2c is a free and open-source lexer generator for C and C++. Its main goal is generating
fast lexers: at least as fast as their reasonably optimized hand-coded counterparts. Instead of using
traditional table-driven approach, re2c encodes the generated finite state automata directly in the form
of conditional jumps and comparisons.
<strong>Homepage:</strong> https://re2c.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>re2c/1.2.1-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###Reapr
<strong>Description:</strong> A tool that evaluates the accuracy of a genome assembly using mapped paired end reads,
without the use of a reference genome for comparison.
<strong>Homepage:</strong> https://www.sanger.ac.uk/science/tools/reapr</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Reapr/1.0.18-foss-2018b</p></li>
<li><p>Reapr/1.0.18-foss-2019a</p></li>
</ul>
</details>
</details>
<p>###RECON
<strong>Description:</strong> De Novo Repeat Finder.
<strong>Homepage:</strong> http://www.repeatmasker.org/RepeatModeler/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>RECON/1.08-foss-2019a</p></li>
</ul>
</details>
</details>
<p>###Relate
<strong>Description:</strong> Software to estimate genome-wide genealogies for thousands of samples.
<strong>Homepage:</strong> https://myersgroup.github.io/relate/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Relate/1.0.17</p></li>
</ul>
</details>
</details>
<p>###RepeatMasker
<strong>Description:</strong> RepeatMasker is a program that screens DNA sequences for interspersed repeats
and low complexity DNA sequences.
<strong>Homepage:</strong> http://www.repeatmasker.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>RepeatMasker/4.0.9-p2-gompi-2019a-HMMER</p></li>
</ul>
</details>
</details>
<p>###RepeatModeler
<strong>Description:</strong> A de novo transposable element (TE) family identification and modeling package.
<strong>Homepage:</strong> http://www.repeatmasker.org/RepeatModeler/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>RepeatModeler/2.0.1-foss-2019a</p></li>
</ul>
</details>
</details>
<p>###RepeatScout
<strong>Description:</strong> De Novo Repeat Finder.
<strong>Homepage:</strong> http://www.repeatmasker.org/RepeatModeler/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>RepeatScout/1.0.6-foss-2019a</p></li>
</ul>
</details>
</details>
<p>###RevBayes
<strong>Description:</strong> RevBayes provides an interactive environment for statistical computation in phylogenetics
<strong>Homepage:</strong> https://revbayes.github.io</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>RevBayes/1.0.12-foss-2019b</p></li>
</ul>
</details>
</details>
<p>###RMBlast
<strong>Description:</strong> RMBlast is a RepeatMasker compatible version of the standard NCBI BLAST suite. The primary difference between
this distribution and the NCBI distribution is the addition of a new program ‘rmblastn’ for use with RepeatMasker and
RepeatModeler.
<strong>Homepage:</strong> http://www.repeatmasker.org/RMBlast.html</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>RMBlast/2.10.0-foss-2019a</p></li>
</ul>
</details>
</details>
<p>###RNAmmer
<strong>Description:</strong> RNAmmer  predicts ribosomal RNA genes in full genome sequences
<strong>Homepage:</strong> http://www.cbs.dtu.dk/services/RNAmmer/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>RNAmmer/1.2-GCC-7.3.0-2.30</p></li>
</ul>
</details>
</details>
<p>###ROHan
<strong>Description:</strong> ROHan is a Bayesian framework to estimate local rates of heterozygosity, infer runs of homozygosity
(ROH) and compute global rates of heterozygosity.
<strong>Homepage:</strong> https://github.com/grenaud/ROHan</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ROHan/200110-GCC-8.2.0-2.31.1</p></li>
</ul>
</details>
</details>
<p>###RSEM
<strong>Description:</strong> RNA-Seq by Expectation-Maximization
<strong>Homepage:</strong> http://deweylab.github.io/RSEM/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>RSEM/1.3.2-foss-2018b</p></li>
</ul>
</details>
</details>
<p>###Ruby
<strong>Description:</strong> Ruby is a dynamic, open source programming language with
a focus on simplicity and productivity. It has an elegant syntax that is
natural to read and easy to write.
<strong>Homepage:</strong> https://www.ruby-lang.org</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Ruby/2.6.3-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###Salmon
<strong>Description:</strong> Salmon is a wicked-fast program to produce a highly-accurate,
transcript-level quantification estimates from RNA-seq data.
<strong>Homepage:</strong> https://github.com/COMBINE-lab/salmon</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Salmon/1.0.0-gompi-2019a</p></li>
</ul>
</details>
</details>
<p>###SAMtools
<strong>Description:</strong> SAM Tools provide various utilities for manipulating alignments in the SAM format,
including sorting, merging, indexing and generating alignments in a per-position format.
<strong>Homepage:</strong> http://www.htslib.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>SAMtools/1.9-GCC-7.3.0-2.30</p></li>
<li><p>SAMtools/1.9-GCC-8.2.0-2.31.1</p></li>
<li><p>SAMtools/1.9-foss-2018b</p></li>
<li><p>SAMtools/1.9-iccifort-2019.1.144-GCC-8.2.0-2.31.1</p></li>
<li><p>SAMtools/1.9-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###ScaLAPACK
<strong>Description:</strong> The ScaLAPACK (or Scalable LAPACK) library includes a subset of LAPACK routines
redesigned for distributed memory MIMD parallel computers.
<strong>Homepage:</strong> http://www.netlib.org/scalapack/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ScaLAPACK/2.0.2-gompi-2018b-OpenBLAS-0.3.1</p></li>
<li><p>ScaLAPACK/2.0.2-gompi-2019a-OpenBLAS-0.3.5</p></li>
<li><p>ScaLAPACK/2.0.2-gompi-2019b</p></li>
<li><p>ScaLAPACK/2.0.2-gompic-2018b-OpenBLAS-0.3.1</p></li>
</ul>
</details>
</details>
<p>###scikit-allel
<strong>Description:</strong> This package provides utilities for exploratory analysis of large scale genetic variation data.
It is based on numpy, scipy and other general-purpose Python scientific libraries.
<strong>Homepage:</strong> http://scikit-allel.readthedocs.io/en/latest/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>scikit-allel/1.2.1-foss-2019a-Python-3.7.2</p></li>
</ul>
</details>
</details>
<p>###scikit-learn
<strong>Description:</strong> Scikit-learn integrates machine learning algorithms in the tightly-knit scientific Python world,
building upon numpy, scipy, and matplotlib. As a machine-learning module,
it provides versatile tools for data mining and analysis in any field of science and engineering.
It strives to be simple and efficient, accessible to everybody, and reusable in various contexts.
<strong>Homepage:</strong> http://scikit-learn.org/stable/index.html</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>scikit-learn/0.20.3-foss-2019a</p></li>
</ul>
</details>
</details>
<p>###SciPy-bundle
<strong>Description:</strong> Bundle of Python packages for scientific software
<strong>Homepage:</strong> https://python.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>SciPy-bundle/2019.03-foss-2019a</p></li>
<li><p>SciPy-bundle/2019.03-intel-2019a</p></li>
<li><p>SciPy-bundle/2019.10-foss-2019b-Python-3.7.4</p></li>
<li><p>SciPy-bundle/2019.10-intel-2019b-Python-3.7.4</p></li>
</ul>
</details>
</details>
<p>###SCons
<strong>Description:</strong> SCons is a software construction tool.
<strong>Homepage:</strong> https://www.scons.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>SCons/3.0.1-GCCcore-7.3.0-Python-2.7.15-bare</p></li>
<li><p>SCons/3.0.4-foss-2018b-Python-2.7.15</p></li>
<li><p>SCons/3.0.5-GCCcore-8.2.0</p></li>
<li><p>SCons/3.1.1-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###SCOTCH
<strong>Description:</strong> Software package and libraries for sequential and parallel graph partitioning,
static mapping, and sparse matrix block ordering, and sequential mesh and hypergraph partitioning.
<strong>Homepage:</strong> http://gforge.inria.fr/projects/scotch/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>SCOTCH/6.0.6-foss-2018b</p></li>
<li><p>SCOTCH/6.0.6-gompi-2019a</p></li>
<li><p>SCOTCH/6.0.6-iimpi-2019a</p></li>
<li><p>SCOTCH/6.0.6-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###Seaborn
<strong>Description:</strong> Seaborn is a Python visualization library based on matplotlib.
It provides a high-level interface for drawing attractive statistical graphics.
<strong>Homepage:</strong> http://stanford.edu/~mwaskom/software/seaborn/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Seaborn/0.9.0-foss-2019a-Python-3.7.2</p></li>
</ul>
</details>
</details>
<p>###SeqKit
<strong>Description:</strong> SeqKit
<strong>Homepage:</strong> https://github.com/shenwei356/seqkit</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>SeqKit/0.11.0</p></li>
</ul>
</details>
</details>
<p>###seqtk
<strong>Description:</strong> Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format.
It seamlessly parses both FASTA and FASTQ files which can also be optionally compressed by gzip.
<strong>Homepage:</strong> https://github.com/lh3/seqtk/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>seqtk/1.3-foss-2018b</p></li>
</ul>
</details>
</details>
<p>###Serf
<strong>Description:</strong> The serf library is a high performance C-based HTTP client library
built upon the Apache Portable Runtime (APR) library
<strong>Homepage:</strong> http://serf.apache.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Serf/1.3.9-GCCcore-7.3.0</p></li>
<li><p>Serf/1.3.9-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###SGA
<strong>Description:</strong> Assembles large genomes from high coverage short read data.
<strong>Homepage:</strong> https://omictools.com/sga-tool</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>SGA/190808-foss-2018b-Python-3.6.6</p></li>
</ul>
</details>
</details>
<p>###SHAPEIT
<strong>Description:</strong> SHAPEIT is a fast and accurate method for estimation of
haplotypes (aka phasing) from genotype or sequencing data.
<strong>Homepage:</strong> https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>SHAPEIT/2.r904.glibcv2.17</p></li>
</ul>
</details>
</details>
<p>###SHAPEIT4
<strong>Description:</strong> SHAPEIT4 is a method for estimation of haplotypes (aka phasing) for SNP array and sequencing data.
<strong>Homepage:</strong> https://github.com/odelaneau/shapeit4</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>SHAPEIT4/4.1-gompi-2019a</p></li>
</ul>
</details>
</details>
<p>###SLEPc
<strong>Description:</strong> SLEPc (Scalable Library for Eigenvalue Problem Computations) is a software library for the solution
of large scale sparse eigenvalue problems on parallel computers. It is an extension of PETSc and can be used for
either standard or generalized eigenproblems, with real or complex arithmetic. It can also be used for computing a
partial SVD of a large, sparse, rectangular matrix, and to solve quadratic eigenvalue problems.
<strong>Homepage:</strong> http://www.grycap.upv.es/slepc/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>SLEPc/3.11.0-foss-2018b</p></li>
<li><p>SLEPc/3.11.0-intel-2018b</p></li>
<li><p>SLEPc/3.11.1-foss-2018b</p></li>
<li><p>SLEPc/3.11.1-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###SNAP
<strong>Description:</strong> SNAP is a general purpose gene finding program suitable for both eukaryotic and prokaryotic genomes.
SNAP is an acroynm for Semi-HMM-based Nucleic Acid Parser.
<strong>Homepage:</strong> http://korflab.ucdavis.edu/software.html</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>SNAP/2013-11-29-GCC-8.2.0-2.31.1</p></li>
<li><p>SNAP/2013-11-29-GCC-8.3.0</p></li>
</ul>
</details>
</details>
<p>###snappy
<strong>Description:</strong> Snappy is a compression/decompression library. It does not aim
for maximum compression, or compatibility with any other compression library;
instead, it aims for very high speeds and reasonable compression.
<strong>Homepage:</strong> https://github.com/google/snappy</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>snappy/1.1.7-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###SOAPdenovo2
<strong>Description:</strong> SOAPdenovo is a novel short-read assembly method that can build a
de novo draft assembly for human-sized genomes. The program is specially designed to
assemble Illumina short reads. It creates new opportunities for building reference
sequences and carrying out accurate analyses of unexplored genomes in a cost effective way.
SOAPdenovo2 is the successor of SOAPdenovo.
<strong>Homepage:</strong> http://soap.genomics.org.cn/index.html</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>SOAPdenovo2/r241-foss-2018b</p></li>
<li><p>SOAPdenovo2/r241-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###SPAdes
<strong>Description:</strong> Genome assembler for single-cell and isolates data sets
<strong>Homepage:</strong> http://cab.spbu.ru/software/spades/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>SPAdes/3.13.0-foss-2018b</p></li>
<li><p>SPAdes/3.13.1-GCC-8.2.0-2.31.1</p></li>
</ul>
</details>
</details>
<p>###sparsehash
<strong>Description:</strong> An extremely memory-efficient hash_map implementation. 2 bits/entry overhead!
The SparseHash library contains several hash-map implementations, including
implementations that optimize for space or speed.</p>
<p><strong>Homepage:</strong> https://github.com/sparsehash/sparsehash</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>sparsehash/2.0.3-GCCcore-7.3.0</p></li>
<li><p>sparsehash/2.0.3-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###Sphinx
<strong>Description:</strong> Sphinx is a tool that makes it easy to create intelligent and beautiful documentation.
It was originally created for the new Python documentation, and it has excellent facilities for the
documentation of Python projects, but C/C++ is already supported as well, and it is planned to add
special support for other languages as well.
<strong>Homepage:</strong> http://sphinx.pocoo.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Sphinx/1.8.1-foss-2018b-Python-2.7.15</p></li>
</ul>
</details>
</details>
<p>###SQLite
<strong>Description:</strong> SQLite: SQL Database Engine in a C Library
<strong>Homepage:</strong> https://www.sqlite.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>SQLite/3.20.1-GCCcore-7.3.0</p></li>
<li><p>SQLite/3.24.0-GCCcore-7.3.0</p></li>
<li><p>SQLite/3.27.2-GCCcore-8.2.0</p></li>
<li><p>SQLite/3.29.0-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###SRA-Toolkit
<strong>Description:</strong> The SRA Toolkit, and the source-code SRA System Development
Kit (SDK), will allow you to programmatically access data housed within SRA
and convert it from the SRA format
<strong>Homepage:</strong> http://www.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?view=software</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>SRA-Toolkit/2.9.6-1-centos_linux64</p></li>
</ul>
</details>
</details>
<p>###Stacks
<strong>Description:</strong> Stacks is a software pipeline for building loci from short-read sequences, such as those generated on
the Illumina platform. Stacks was developed to work with restriction enzyme-based data, such as RAD-seq,
for the purpose of building genetic maps and conducting population genomics and phylogeography.</p>
<p><strong>Homepage:</strong> http://catchenlab.life.illinois.edu/stacks</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Stacks/1.48-intel-2018b</p></li>
<li><p>Stacks/2.3e-foss-2018b</p></li>
<li><p>Stacks/2.41-GCC-8.2.0-2.31.1</p></li>
<li><p>Stacks/2.41-iccifort-2019.1.144-GCC-8.2.0-2.31.1</p></li>
</ul>
</details>
</details>
<p>###Stampy
<strong>Description:</strong> Stampy is a package for the mapping of short reads from illumina sequencing machines onto a reference genome
<strong>Homepage:</strong> https://www.well.ox.ac.uk/research/research-groups/lunter-group/lunter-group/stampy</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Stampy/1.0.32-foss-2018b-Python-2.7.15</p></li>
</ul>
</details>
</details>
<p>###StringTie
<strong>Description:</strong> StringTie is a fast and highly efficient assembler of RNA-Seq alignments into potential transcripts.
<strong>Homepage:</strong> http://ccb.jhu.edu/software/%(namelower)/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>StringTie/1.3.5-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###Structure
<strong>Description:</strong> The program structure is a free software package for using multi-locus genotype data to investigate
population structure.
<strong>Homepage:</strong> https://web.stanford.edu/group/pritchardlab/structure.html</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Structure/2.3.4-GCC-7.3.0-2.30</p></li>
</ul>
</details>
</details>
<p>###Subversion
<strong>Description:</strong> Subversion is an open source version control system.
<strong>Homepage:</strong> http://subversion.apache.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Subversion/1.10.2-GCCcore-7.3.0</p></li>
<li><p>Subversion/1.12.0-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###SuiteSparse
<strong>Description:</strong> SuiteSparse is a collection of libraries manipulate sparse matrices.
<strong>Homepage:</strong> http://faculty.cse.tamu.edu/davis/suitesparse.html</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>SuiteSparse/5.4.0-foss-2018b-METIS-5.1.0</p></li>
<li><p>SuiteSparse/5.4.0-foss-2019a-METIS-5.1.0</p></li>
<li><p>SuiteSparse/5.4.0-intel-2018b-METIS-5.1.0</p></li>
<li><p>SuiteSparse/5.4.0-intel-2019a-METIS-5.1.0</p></li>
</ul>
</details>
</details>
<p>###Sumaclust
<strong>Description:</strong> SUMACLUST is a sequence-clustering application
<strong>Homepage:</strong> https://git.metabarcoding.org/obitools/sumaclust/wikis/home</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Sumaclust/1.0.34-GCC-7.3.0-2.30</p></li>
</ul>
</details>
</details>
<p>###SUNDIALS
<strong>Description:</strong> SUNDIALS: SUite of Nonlinear and DIfferential/ALgebraic Equation Solvers
<strong>Homepage:</strong> http://computation.llnl.gov/projects/sundials</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>SUNDIALS/2.7.0-foss-2018b</p></li>
</ul>
</details>
</details>
<p>###SWIG
<strong>Description:</strong> SWIG is a software development tool that connects programs written in C and C++ with
a variety of high-level programming languages.
<strong>Homepage:</strong> http://www.swig.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>SWIG/3.0.12-GCCcore-8.2.0-Python-3.7.2</p></li>
<li><p>SWIG/3.0.12-foss-2018b-Python-3.6.6</p></li>
<li><p>SWIG/3.0.12-intel-2018b-Python-3.6.6</p></li>
<li><p>SWIG/4.0.1-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###Szip
<strong>Description:</strong> Szip compression software, providing lossless compression of scientific data</p>
<p><strong>Homepage:</strong> https://www.hdfgroup.org/doc_resource/SZIP/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Szip/2.1.1-GCCcore-7.3.0</p></li>
<li><p>Szip/2.1.1-GCCcore-8.2.0</p></li>
<li><p>Szip/2.1.1-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###tabix
<strong>Description:</strong> Generic indexer for TAB-delimited genome position files
<strong>Homepage:</strong> http://samtools.sourceforge.net</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>tabix/0.2.6-GCCcore-7.3.0</p></li>
</ul>
</details>
</details>
<p>###tbb
<strong>Description:</strong> Intel(R) Threading Building Blocks (Intel(R) TBB) lets you easily write parallel C++ programs that
take full advantage of multicore performance, that are portable, composable and have future-proof scalability.
<strong>Homepage:</strong> https://01.org/tbb/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>tbb/2018_U5-GCCcore-7.3.0</p></li>
<li><p>tbb/2019_U4-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###Tcl
<strong>Description:</strong> Tcl (Tool Command Language) is a very powerful but easy to learn dynamic
programming language, suitable for a very wide range of uses, including web
and desktop applications, networking, administration, testing and many more.</p>
<p><strong>Homepage:</strong> https://www.tcl.tk/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Tcl/8.6.7-GCCcore-7.3.0</p></li>
<li><p>Tcl/8.6.8-GCCcore-7.3.0</p></li>
<li><p>Tcl/8.6.9-GCCcore-8.2.0</p></li>
<li><p>Tcl/8.6.9-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###TensorFlow
<strong>Description:</strong> An open-source software library for Machine Intelligence
<strong>Homepage:</strong> https://www.tensorflow.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>TensorFlow/1.13.1-fosscuda-2018b-Python-3.6.6</p></li>
</ul>
</details>
</details>
<p>###Theano
<strong>Description:</strong> Theano is a Python library that allows you to define, optimize,
and evaluate mathematical expressions involving multi-dimensional arrays efficiently.
<strong>Homepage:</strong> http://deeplearning.net/software/theano</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Theano/1.0.4-fosscuda-2018b-Python-3.6.6</p></li>
</ul>
</details>
</details>
<p>###Tigmint
<strong>Description:</strong> Tigmint identifies and corrects misassemblies using linked reads from 10x Genomics Chromium
<strong>Homepage:</strong> https://github.com/bcgsc/tigmint</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Tigmint/1.1.2-foss-2018b-Python-3.6.6</p></li>
</ul>
</details>
</details>
<p>###Tk
<strong>Description:</strong> Tk is an open source, cross-platform widget toolchain that provides a library of basic elements for
building a graphical user interface (GUI) in many different programming languages.
<strong>Homepage:</strong> https://www.tcl.tk/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Tk/8.6.8-GCCcore-7.3.0</p></li>
<li><p>Tk/8.6.9-GCCcore-8.2.0</p></li>
<li><p>Tk/8.6.9-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###Tkinter
<strong>Description:</strong> Tkinter module, built with the Python buildsystem
<strong>Homepage:</strong> https://python.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Tkinter/2.7.15-GCCcore-8.2.0</p></li>
<li><p>Tkinter/2.7.15-foss-2018b-Python-2.7.15</p></li>
<li><p>Tkinter/2.7.15-fosscuda-2018b-Python-2.7.15</p></li>
<li><p>Tkinter/2.7.15-intel-2018b-Python-2.7.15</p></li>
<li><p>Tkinter/3.6.6-foss-2018b-Python-3.6.6</p></li>
<li><p>Tkinter/3.6.6-intel-2018b-Python-3.6.6</p></li>
<li><p>Tkinter/3.7.2-GCCcore-8.2.0</p></li>
<li><p>Tkinter/3.7.4-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###TotalView
<strong>Description:</strong> TotalView is a GUI-based source code defect analysis tool that gives you unprecedented
control over processes and thread execution and visibility into program state and variables. It allows
you to debug one or many processes and/or threads in a single window with complete control over program
execution. This allows you to set breakpoints, stepping line by line through the code on a single thread,
or with coordinated groups of processes or threads, and run or halt arbitrary sets of processes or threads.
You can reproduce and troubleshoot difficult problems that can occur in concurrent programs that take
advantage of threads, OpenMP, MPI, GPUs or coprocessors.
<strong>Homepage:</strong> http://www.roguewave.com/products/totalview.aspx</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>TotalView/2019.1.4</p></li>
</ul>
</details>
</details>
<p>###TransDecoder
<strong>Description:</strong> TransDecoder identifies candidate coding regions within transcript sequences,
such as those generated by de novo RNA-Seq transcript assembly using Trinity,
or constructed based on RNA-Seq alignments to the genome using
Tophat and Cufflinks.
<strong>Homepage:</strong> https://github.com/TransDecoder/TransDecoder/wiki</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>TransDecoder/5.5.0-intel-2018b-Perl-5.28.0</p></li>
</ul>
</details>
</details>
<p>###TreeMix
<strong>Description:</strong> TreeMix is a method for inferring the patterns of population splits and mixtures in the history of a
set of populations.
<strong>Homepage:</strong> http://bitbucket.org/nygcresearch/treemix</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>TreeMix/1.13-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###TRF
<strong>Description:</strong> Tandem repeats finder: a program to analyze DNA sequences. Legacy version.
<strong>Homepage:</strong> https://tandem.bu.edu/trf/trf.html</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>TRF/4.09-linux64</p></li>
</ul>
</details>
</details>
<p>###Trilinos
<strong>Description:</strong> The Trilinos Project is an effort to develop algorithms and enabling technologies
within an object-oriented software framework for the solution of large-scale, complex multi-physics
engineering and scientific problems. A unique design feature of Trilinos is its focus on packages.
<strong>Homepage:</strong> https://trilinos.org</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Trilinos/12.12.1-foss-2018b-Python-3.6.6</p></li>
<li><p>Trilinos/12.12.1-intel-2018b-Python-3.6.6</p></li>
</ul>
</details>
</details>
<p>###Trim_Galore
<strong>Description:</strong> Trim Galore is a wrapper around Cutadapt and FastQC to
consistently apply adapter and quality trimming to FastQ files, with extra
functionality for RRBS data.
<strong>Homepage:</strong> https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Trim_Galore/0.6.2-foss-2018b-Python-3.6.6</p></li>
</ul>
</details>
</details>
<p>###Trimmomatic
<strong>Description:</strong> Trimmomatic performs a variety of useful trimming tasks for illumina
paired-end and single ended data.The selection of trimming steps and their associated
parameters are supplied on the command line.
<strong>Homepage:</strong> http://www.usadellab.org/cms/?page=trimmomatic</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Trimmomatic/0.38-Java-1.8</p></li>
</ul>
</details>
</details>
<p>###Trinity
<strong>Description:</strong> Trinity represents a novel method for the efficient and robust de novo reconstruction
of transcriptomes from RNA-Seq data. Trinity combines three independent software modules: Inchworm,
Chrysalis, and Butterfly, applied sequentially to process large volumes of RNA-Seq reads.
<strong>Homepage:</strong> http://trinityrnaseq.github.io</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Trinity/2.8.4-foss-2018b</p></li>
</ul>
</details>
</details>
<p>###UDUNITS
<strong>Description:</strong> UDUNITS supports conversion of unit specifications between formatted and binary forms,
arithmetic manipulation of units, and conversion of values between compatible scales of measurement.
<strong>Homepage:</strong> http://www.unidata.ucar.edu/software/udunits/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>UDUNITS/2.2.26-GCCcore-8.2.0</p></li>
<li><p>UDUNITS/2.2.26-GCCcore-8.3.0</p></li>
<li><p>UDUNITS/2.2.26-foss-2018b</p></li>
<li><p>UDUNITS/2.2.26-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###UFL
<strong>Description:</strong> The Unified Form Language (UFL) is a domain specific language
for declaration of finite element discretizations of variational forms.
More precisely, it defines a flexible interface for choosing finite element
spaces and defining expressions for weak forms in a notation close to
mathematical notation.
<strong>Homepage:</strong> https://bitbucket.org/fenics-project/ufl</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>UFL/2018.1.0-foss-2018b-Python-3.6.6</p></li>
<li><p>UFL/2019.1.0-foss-2019a-Python-3.7.2</p></li>
</ul>
</details>
</details>
<p>###USEARCH
<strong>Description:</strong> USEARCH is a unique sequence analysis tool which offers search and clustering algorithms that are
often orders of magnitude faster than BLAST.
<strong>Homepage:</strong> http://www.drive5.com/usearch/index.html</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>USEARCH/11.0.667-i86linux32</p></li>
</ul>
</details>
</details>
<p>###utf8proc
<strong>Description:</strong> utf8proc is a small, clean C library that provides Unicode normalization, case-folding,
and other operations for data in the UTF-8 encoding.
<strong>Homepage:</strong> https://github.com/JuliaStrings/utf8proc</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>utf8proc/2.3.0-GCCcore-8.2.0</p></li>
</ul>
</details>
</details>
<p>###util-linux
<strong>Description:</strong> Set of Linux utilities
<strong>Homepage:</strong> https://www.kernel.org/pub/linux/utils/util-linux</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>util-linux/2.32-GCCcore-7.3.0</p></li>
<li><p>util-linux/2.33-GCCcore-8.2.0</p></li>
<li><p>util-linux/2.34-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###Valgrind
<strong>Description:</strong> Valgrind: Debugging and profiling tools
<strong>Homepage:</strong> http://valgrind.org</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Valgrind/3.14.0-foss-2018b</p></li>
<li><p>Valgrind/3.14.0-intel-2018b</p></li>
</ul>
</details>
</details>
<p>###VASP
<strong>Description:</strong> The Vienna Ab initio Simulation Package (VASP) is a computer program for atomic scale materials modelling, e.g. electronic structure calculations and quantum-mechanical molecular dynamics, from first principles.
<strong>Homepage:</strong> http://www.vasp.at</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>VASP/5.4.4-intel-2019a-gam</p></li>
<li><p>VASP/5.4.4-intel-2019a-gam-vtst</p></li>
<li><p>VASP/5.4.4-intel-2019a-ncl</p></li>
<li><p>VASP/5.4.4-intel-2019a-std</p></li>
<li><p>VASP/5.4.4-intel-2019a-std-vtst</p></li>
</ul>
</details>
</details>
<p>###VCFtools
<strong>Description:</strong> The aim of VCFtools is to provide
easily accessible methods for working with complex
genetic variation data in the form of VCF files.
<strong>Homepage:</strong> https://vcftools.github.io</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>VCFtools/0.1.16-foss-2018b-Perl-5.28.0</p></li>
<li><p>VCFtools/0.1.16-intel-2018b-Perl-5.28.0</p></li>
</ul>
</details>
</details>
<p>###Velvet
<strong>Description:</strong> Sequence assembler for very short reads
<strong>Homepage:</strong> http://www.ebi.ac.uk/~zerbino/velvet/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Velvet/1.2.10-foss-2018b-mt-kmer_191</p></li>
<li><p>Velvet/1.2.10-intel-2018b-mt-kmer_191</p></li>
</ul>
</details>
</details>
<p>###VISLCG3
<strong>Description:</strong> Implements the functionality of the VISL CG-2 variant of constraint grammar.
<strong>Homepage:</strong> https://visl.sdu.dk/cg3.html</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>VISLCG3/1.3.0-foss-2019a</p></li>
</ul>
</details>
</details>
<p>###VSEARCH
<strong>Description:</strong> VSEARCH supports de novo and reference based chimera detection,
clustering, full-length and prefix dereplication, rereplication,
reverse complementation, masking, all-vs-all pairwise global alignment,
exact and global alignment searching, shuffling, subsampling and sorting.
It also supports FASTQ file analysis, filtering,
conversion and merging of paired-end reads.
<strong>Homepage:</strong> https://github.com/torognes/vsearch</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>VSEARCH/2.9.1-foss-2018b</p></li>
<li><p>VSEARCH/2.13.4-iccifort-2019.1.144-GCC-8.2.0-2.31.1</p></li>
</ul>
</details>
</details>
<p>###VTK
<strong>Description:</strong> The Visualization Toolkit (VTK) is an open-source, freely available software system for
3D computer graphics, image processing and visualization. VTK consists of a C++ class library and several
interpreted interface layers including Tcl/Tk, Java, and Python. VTK supports a wide variety of visualization
algorithms including: scalar, vector, tensor, texture, and volumetric methods; and advanced modeling techniques
such as: implicit modeling, polygon reduction, mesh smoothing, cutting, contouring, and Delaunay triangulation.
<strong>Homepage:</strong> https://www.vtk.org</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>VTK/8.1.1-foss-2018b-Python-2.7.15</p></li>
<li><p>VTK/8.1.1-foss-2018b-Python-3.6.6</p></li>
<li><p>VTK/8.2.0-foss-2019a-Python-3.7.2</p></li>
</ul>
</details>
</details>
<p>###VTune
<strong>Description:</strong> Intel VTune Amplifier XE is the premier performance profiler for C, C++, C#, Fortran,
Assembly and Java.
<strong>Homepage:</strong> https://software.intel.com/en-us/vtune</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>VTune/2019_update5</p></li>
</ul>
</details>
</details>
<p>###Wannier90
<strong>Description:</strong> A tool for obtaining maximally-localised Wannier functions
<strong>Homepage:</strong> http://www.wannier.org</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>Wannier90/2.1.0-intel-2019a</p></li>
</ul>
</details>
</details>
<p>###wheel
<strong>Description:</strong> A built-package format for Python.
<strong>Homepage:</strong> https://pypi.python.org/pypi/wheel</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>wheel/0.31.1-foss-2018b-Python-2.7.15</p></li>
<li><p>wheel/0.31.1-foss-2018b-Python-3.6.6</p></li>
<li><p>wheel/0.31.1-fosscuda-2018b-Python-3.6.6</p></li>
</ul>
</details>
</details>
<p>###X11
<strong>Description:</strong> The X Window System (X11) is a windowing system for bitmap displays
<strong>Homepage:</strong> https://www.x.org</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>X11/20180604-GCCcore-7.3.0</p></li>
<li><p>X11/20190311-GCCcore-8.2.0</p></li>
<li><p>X11/20190717-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###XML-LibXML
<strong>Description:</strong> Perl binding for libxml2
<strong>Homepage:</strong> https://metacpan.org/pod/distribution/XML-LibXML/LibXML.pod</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>XML-LibXML/2.0132-GCCcore-7.3.0-Perl-5.28.0</p></li>
<li><p>XML-LibXML/2.0200-GCCcore-8.2.0-Perl-5.28.1</p></li>
<li><p>XML-LibXML/2.0201-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###XML-Parser
<strong>Description:</strong> This is a Perl extension interface to James Clark’s XML parser, expat.
<strong>Homepage:</strong> http://search.cpan.org/~toddr/XML-Parser-2.41/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>XML-Parser/2.44_01-GCCcore-7.3.0-Perl-5.28.0</p></li>
</ul>
</details>
</details>
<p>###xorg-macros
<strong>Description:</strong> X.org macros utilities.
<strong>Homepage:</strong> https://cgit.freedesktop.org/xorg/util/macros</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>xorg-macros/1.19.2-GCCcore-7.3.0</p></li>
<li><p>xorg-macros/1.19.2-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###XZ
<strong>Description:</strong> xz: XZ utilities
<strong>Homepage:</strong> https://tukaani.org/xz/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>XZ/5.2.4-GCCcore-7.3.0</p></li>
<li><p>XZ/5.2.4-GCCcore-8.2.0</p></li>
<li><p>XZ/5.2.4-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>
<p>###YAXT
<strong>Description:</strong> Yet Another eXchange Tool
<strong>Homepage:</strong> https://www.dkrz.de/redmine/projects/yaxt</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>YAXT/0.6.0-intel-2018b</p></li>
<li><p>YAXT/0.6.2-iimpi-2019b</p></li>
</ul>
</details>
</details>
<p>###zarr
<strong>Description:</strong> Zarr is a Python package providing an implementation of compressed, chunked, N-dimensional arrays,
designed for use in parallel computing.
<strong>Homepage:</strong> http://zarr.readthedocs.io/en/stable/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>zarr/2.3.1-foss-2019a</p></li>
</ul>
</details>
</details>
<p>###ZeroMQ
<strong>Description:</strong> ZeroMQ looks like an embeddable networking library but acts like a concurrency framework.
It gives you sockets that carry atomic messages across various transports like in-process,
inter-process, TCP, and multicast. You can connect sockets N-to-N with patterns like fanout,
pub-sub, task distribution, and request-reply. It’s fast enough to be the fabric for clustered
products. Its asynchronous I/O model gives you scalable multicore applications, built as asynchronous
message-processing tasks. It has a score of language APIs and runs on most operating systems.
<strong>Homepage:</strong> http://www.zeromq.org/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>ZeroMQ/4.2.5-foss-2018b</p></li>
</ul>
</details>
</details>
<p>###zlib
<strong>Description:</strong> zlib is designed to be a free, general-purpose, legally unencumbered – that
is, not covered by any patents – lossless data-compression library for use
on virtually any computer hardware and operating system.</p>
<p><strong>Homepage:</strong> http://www.zlib.net/</p>
<details><summary> <u>Installed versions:</u> </summary>
<ul class="simple">
<li><p>zlib/1.2.11</p></li>
<li><p>zlib/1.2.11-GCCcore-6.4.0</p></li>
<li><p>zlib/1.2.11-GCCcore-7.3.0</p></li>
<li><p>zlib/1.2.11-GCCcore-8.2.0</p></li>
<li><p>zlib/1.2.11-GCCcore-8.3.0</p></li>
</ul>
</details>
</details>


           </div>
           
          </div>
          <footer>
  

  <hr/>

  <div role="contentinfo">
    <p>
        
        &copy; Copyright 2020, Sigma2/Metacenter

    </p>
  </div>
    
    
    
    Built with <a href="http://sphinx-doc.org/">Sphinx</a> using a
    
    <a href="https://github.com/rtfd/sphinx_rtd_theme">theme</a>
    
    provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>

        </div>
      </div>

    </section>

  </div>
  

  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script>

  
  
    
   

</body>
</html>